// lucide-svelte/dist/aliases/aliases.js
/**
 * @license lucide-svelte v0.456.0 - ISC
 *
 * This source code is licensed under the ISC license.
 * See the LICENSE file in the root directory of this source tree.
 */
// AlarmClockCheck aliases
export { default as AlarmCheck } from '../icons/alarm-clock-check.svelte';
// AlarmClockMinus aliases
export { default as AlarmMinus } from '../icons/alarm-clock-minus.svelte';
// AlarmClockPlus aliases
export { default as AlarmPlus } from '../icons/alarm-clock-plus.svelte';
// ArrowDownAZ aliases
export { default as ArrowDownAz } from '../icons/arrow-down-a-z.svelte';
// ArrowDownWideNarrow aliases
export { default as SortDesc } from '../icons/arrow-down-wide-narrow.svelte';
// ArrowDownZA aliases
export { default as ArrowDownZa } from '../icons/arrow-down-z-a.svelte';
// ArrowUpAZ aliases
export { default as ArrowUpAz } from '../icons/arrow-up-a-z.svelte';
// ArrowUpNarrowWide aliases
export { default as SortAsc } from '../icons/arrow-up-narrow-wide.svelte';
// ArrowUpZA aliases
export { default as ArrowUpZa } from '../icons/arrow-up-z-a.svelte';
// Axis3d aliases
export { default as Axis3D } from '../icons/axis-3d.svelte';
// BadgeCheck aliases
export { default as Verified } from '../icons/badge-check.svelte';
// BetweenHorizontalEnd aliases
export { 
/** @deprecated Renamed because of typo, use {@link BetweenHorizontalEnd} instead. This alias will be removed in v1.0 */
default as BetweenHorizonalEnd } from '../icons/between-horizontal-end.svelte';
// BetweenHorizontalStart aliases
export { 
/** @deprecated Renamed because of typo, use {@link BetweenHorizontalStart} instead. This alias will be removed in v1.0 */
default as BetweenHorizonalStart } from '../icons/between-horizontal-start.svelte';
// BookDashed aliases
export { default as BookTemplate } from '../icons/book-dashed.svelte';
// Braces aliases
export { default as CurlyBraces } from '../icons/braces.svelte';
// Captions aliases
export { default as Subtitles } from '../icons/captions.svelte';
// ChartArea aliases
export { 
/** @deprecated  */
default as AreaChart } from '../icons/chart-area.svelte';
// ChartBarBig aliases
export { 
/** @deprecated  */
default as BarChartHorizontalBig } from '../icons/chart-bar-big.svelte';
// ChartBar aliases
export { 
/** @deprecated  */
default as BarChartHorizontal } from '../icons/chart-bar.svelte';
// ChartCandlestick aliases
export { 
/** @deprecated  */
default as CandlestickChart } from '../icons/chart-candlestick.svelte';
// ChartColumnBig aliases
export { 
/** @deprecated  */
default as BarChartBig } from '../icons/chart-column-big.svelte';
// ChartColumnIncreasing aliases
export { 
/** @deprecated  */
default as BarChart4 } from '../icons/chart-column-increasing.svelte';
// ChartColumn aliases
export { 
/** @deprecated  */
default as BarChart3 } from '../icons/chart-column.svelte';
// ChartLine aliases
export { 
/** @deprecated  */
default as LineChart } from '../icons/chart-line.svelte';
// ChartNoAxesColumnIncreasing aliases
export { 
/** @deprecated  */
default as BarChart } from '../icons/chart-no-axes-column-increasing.svelte';
// ChartNoAxesColumn aliases
export { 
/** @deprecated  */
default as BarChart2 } from '../icons/chart-no-axes-column.svelte';
// ChartNoAxesGantt aliases
export { 
/** @deprecated  */
default as GanttChart } from '../icons/chart-no-axes-gantt.svelte';
// ChartPie aliases
export { 
/** @deprecated  */
default as PieChart } from '../icons/chart-pie.svelte';
// ChartScatter aliases
export { 
/** @deprecated  */
default as ScatterChart } from '../icons/chart-scatter.svelte';
// CircleAlert aliases
export { default as AlertCircle } from '../icons/circle-alert.svelte';
// CircleArrowDown aliases
export { default as ArrowDownCircle } from '../icons/circle-arrow-down.svelte';
// CircleArrowLeft aliases
export { default as ArrowLeftCircle } from '../icons/circle-arrow-left.svelte';
// CircleArrowOutDownLeft aliases
export { default as ArrowDownLeftFromCircle } from '../icons/circle-arrow-out-down-left.svelte';
// CircleArrowOutDownRight aliases
export { default as ArrowDownRightFromCircle } from '../icons/circle-arrow-out-down-right.svelte';
// CircleArrowOutUpLeft aliases
export { default as ArrowUpLeftFromCircle } from '../icons/circle-arrow-out-up-left.svelte';
// CircleArrowOutUpRight aliases
export { default as ArrowUpRightFromCircle } from '../icons/circle-arrow-out-up-right.svelte';
// CircleArrowRight aliases
export { default as ArrowRightCircle } from '../icons/circle-arrow-right.svelte';
// CircleArrowUp aliases
export { default as ArrowUpCircle } from '../icons/circle-arrow-up.svelte';
// CircleCheckBig aliases
export { default as CheckCircle } from '../icons/circle-check-big.svelte';
// CircleCheck aliases
export { default as CheckCircle2 } from '../icons/circle-check.svelte';
// CircleChevronDown aliases
export { default as ChevronDownCircle } from '../icons/circle-chevron-down.svelte';
// CircleChevronLeft aliases
export { default as ChevronLeftCircle } from '../icons/circle-chevron-left.svelte';
// CircleChevronRight aliases
export { default as ChevronRightCircle } from '../icons/circle-chevron-right.svelte';
// CircleChevronUp aliases
export { default as ChevronUpCircle } from '../icons/circle-chevron-up.svelte';
// CircleDivide aliases
export { default as DivideCircle } from '../icons/circle-divide.svelte';
// CircleGauge aliases
export { default as GaugeCircle } from '../icons/circle-gauge.svelte';
// CircleHelp aliases
export { default as HelpCircle } from '../icons/circle-help.svelte';
// CircleMinus aliases
export { default as MinusCircle } from '../icons/circle-minus.svelte';
// CircleParkingOff aliases
export { default as ParkingCircleOff } from '../icons/circle-parking-off.svelte';
// CircleParking aliases
export { default as ParkingCircle } from '../icons/circle-parking.svelte';
// CirclePause aliases
export { default as PauseCircle } from '../icons/circle-pause.svelte';
// CirclePercent aliases
export { default as PercentCircle } from '../icons/circle-percent.svelte';
// CirclePlay aliases
export { default as PlayCircle } from '../icons/circle-play.svelte';
// CirclePlus aliases
export { default as PlusCircle } from '../icons/circle-plus.svelte';
// CirclePower aliases
export { default as PowerCircle } from '../icons/circle-power.svelte';
// CircleSlash2 aliases
export { default as CircleSlashed } from '../icons/circle-slash-2.svelte';
// CircleStop aliases
export { default as StopCircle } from '../icons/circle-stop.svelte';
// CircleUserRound aliases
export { 
/** @deprecated  */
default as UserCircle2 } from '../icons/circle-user-round.svelte';
// CircleUser aliases
export { default as UserCircle } from '../icons/circle-user.svelte';
// CircleX aliases
export { default as XCircle } from '../icons/circle-x.svelte';
// ClipboardPenLine aliases
export { default as ClipboardSignature } from '../icons/clipboard-pen-line.svelte';
// ClipboardPen aliases
export { default as ClipboardEdit } from '../icons/clipboard-pen.svelte';
// CloudDownload aliases
export { default as DownloadCloud } from '../icons/cloud-download.svelte';
// CloudUpload aliases
export { default as UploadCloud } from '../icons/cloud-upload.svelte';
// CodeXml aliases
export { 
/** @deprecated  */
default as Code2 } from '../icons/code-xml.svelte';
// Columns2 aliases
export { default as Columns } from '../icons/columns-2.svelte';
// Columns3 aliases
export { default as PanelsLeftRight } from '../icons/columns-3.svelte';
// ContactRound aliases
export { 
/** @deprecated  */
default as Contact2 } from '../icons/contact-round.svelte';
// DiamondPercent aliases
export { default as PercentDiamond } from '../icons/diamond-percent.svelte';
// Earth aliases
export { 
/** @deprecated  */
default as Globe2 } from '../icons/earth.svelte';
// EllipsisVertical aliases
export { default as MoreVertical } from '../icons/ellipsis-vertical.svelte';
// Ellipsis aliases
export { default as MoreHorizontal } from '../icons/ellipsis.svelte';
// FileAxis3d aliases
export { default as FileAxis3D } from '../icons/file-axis-3d.svelte';
// FileChartColumnIncreasing aliases
export { 
/** @deprecated  */
default as FileBarChart } from '../icons/file-chart-column-increasing.svelte';
// FileChartColumn aliases
export { 
/** @deprecated  */
default as FileBarChart2 } from '../icons/file-chart-column.svelte';
// FileChartLine aliases
export { 
/** @deprecated  */
default as FileLineChart } from '../icons/file-chart-line.svelte';
// FileChartPie aliases
export { 
/** @deprecated  */
default as FilePieChart } from '../icons/file-chart-pie.svelte';
// FileCog aliases
export { 
/** @deprecated  */
default as FileCog2 } from '../icons/file-cog.svelte';
// FilePenLine aliases
export { default as FileSignature } from '../icons/file-pen-line.svelte';
// FilePen aliases
export { default as FileEdit } from '../icons/file-pen.svelte';
// FolderCog aliases
export { 
/** @deprecated  */
default as FolderCog2 } from '../icons/folder-cog.svelte';
// FolderPen aliases
export { default as FolderEdit } from '../icons/folder-pen.svelte';
// GitCommitHorizontal aliases
export { default as GitCommit } from '../icons/git-commit-horizontal.svelte';
// Grid2x2Plus aliases
export { default as Grid2X2Plus } from '../icons/grid-2x2-plus.svelte';
// Grid2x2 aliases
export { default as Grid2X2 } from '../icons/grid-2x2.svelte';
// Grid3x3 aliases
export { default as Grid } from '../icons/grid-3x3.svelte';
// Grid3x3 aliases
export { default as Grid3X3 } from '../icons/grid-3x3.svelte';
// HandHelping aliases
export { default as HelpingHand } from '../icons/hand-helping.svelte';
// House aliases
export { default as Home } from '../icons/house.svelte';
// IceCreamBowl aliases
export { 
/** @deprecated  */
default as IceCream2 } from '../icons/ice-cream-bowl.svelte';
// IceCreamCone aliases
export { default as IceCream } from '../icons/ice-cream-cone.svelte';
// IndentDecrease aliases
export { default as Outdent } from '../icons/indent-decrease.svelte';
// IndentIncrease aliases
export { default as Indent } from '../icons/indent-increase.svelte';
// LaptopMinimal aliases
export { 
/** @deprecated  */
default as Laptop2 } from '../icons/laptop-minimal.svelte';
// LoaderCircle aliases
export { default as Loader2 } from '../icons/loader-circle.svelte';
// LockKeyholeOpen aliases
export { default as UnlockKeyhole } from '../icons/lock-keyhole-open.svelte';
// LockOpen aliases
export { default as Unlock } from '../icons/lock-open.svelte';
// MicVocal aliases
export { 
/** @deprecated  */
default as Mic2 } from '../icons/mic-vocal.svelte';
// Move3d aliases
export { default as Move3D } from '../icons/move-3d.svelte';
// OctagonAlert aliases
export { default as AlertOctagon } from '../icons/octagon-alert.svelte';
// OctagonPause aliases
export { default as PauseOctagon } from '../icons/octagon-pause.svelte';
// OctagonX aliases
export { default as XOctagon } from '../icons/octagon-x.svelte';
// PaintbrushVertical aliases
export { default as Paintbrush2 } from '../icons/paintbrush-vertical.svelte';
// PanelBottomDashed aliases
export { default as PanelBottomInactive } from '../icons/panel-bottom-dashed.svelte';
// PanelLeftClose aliases
export { default as SidebarClose } from '../icons/panel-left-close.svelte';
// PanelLeftDashed aliases
export { default as PanelLeftInactive } from '../icons/panel-left-dashed.svelte';
// PanelLeftOpen aliases
export { default as SidebarOpen } from '../icons/panel-left-open.svelte';
// PanelLeft aliases
export { default as Sidebar } from '../icons/panel-left.svelte';
// PanelRightDashed aliases
export { default as PanelRightInactive } from '../icons/panel-right-dashed.svelte';
// PanelTopDashed aliases
export { default as PanelTopInactive } from '../icons/panel-top-dashed.svelte';
// PanelsTopLeft aliases
export { default as Layout } from '../icons/panels-top-left.svelte';
// PenLine aliases
export { 
/** @deprecated  */
default as Edit3 } from '../icons/pen-line.svelte';
// Pen aliases
export { 
/** @deprecated  */
default as Edit2 } from '../icons/pen.svelte';
// PlugZap aliases
export { default as PlugZap2 } from '../icons/plug-zap.svelte';
// RectangleEllipsis aliases
export { default as FormInput } from '../icons/rectangle-ellipsis.svelte';
// Rotate3d aliases
export { default as Rotate3D } from '../icons/rotate-3d.svelte';
// Rows2 aliases
export { default as Rows } from '../icons/rows-2.svelte';
// Rows3 aliases
export { default as PanelsTopBottom } from '../icons/rows-3.svelte';
// Scale3d aliases
export { default as Scale3D } from '../icons/scale-3d.svelte';
// SendHorizontal aliases
export { 
/** @deprecated Renamed because of typo, use {@link SendHorizontal} instead. This alias will be removed in v1.0 */
default as SendHorizonal } from '../icons/send-horizontal.svelte';
// ShieldX aliases
export { default as ShieldClose } from '../icons/shield-x.svelte';
// SlidersVertical aliases
export { default as Sliders } from '../icons/sliders-vertical.svelte';
// Sparkles aliases
export { default as Stars } from '../icons/sparkles.svelte';
// SquareActivity aliases
export { default as ActivitySquare } from '../icons/square-activity.svelte';
// SquareArrowDownLeft aliases
export { default as ArrowDownLeftSquare } from '../icons/square-arrow-down-left.svelte';
// SquareArrowDownRight aliases
export { default as ArrowDownRightSquare } from '../icons/square-arrow-down-right.svelte';
// SquareArrowDown aliases
export { default as ArrowDownSquare } from '../icons/square-arrow-down.svelte';
// SquareArrowLeft aliases
export { default as ArrowLeftSquare } from '../icons/square-arrow-left.svelte';
// SquareArrowOutDownLeft aliases
export { default as ArrowDownLeftFromSquare } from '../icons/square-arrow-out-down-left.svelte';
// SquareArrowOutDownRight aliases
export { default as ArrowDownRightFromSquare } from '../icons/square-arrow-out-down-right.svelte';
// SquareArrowOutUpLeft aliases
export { default as ArrowUpLeftFromSquare } from '../icons/square-arrow-out-up-left.svelte';
// SquareArrowOutUpRight aliases
export { default as ArrowUpRightFromSquare } from '../icons/square-arrow-out-up-right.svelte';
// SquareArrowRight aliases
export { default as ArrowRightSquare } from '../icons/square-arrow-right.svelte';
// SquareArrowUpLeft aliases
export { default as ArrowUpLeftSquare } from '../icons/square-arrow-up-left.svelte';
// SquareArrowUpRight aliases
export { default as ArrowUpRightSquare } from '../icons/square-arrow-up-right.svelte';
// SquareArrowUp aliases
export { default as ArrowUpSquare } from '../icons/square-arrow-up.svelte';
// SquareAsterisk aliases
export { default as AsteriskSquare } from '../icons/square-asterisk.svelte';
// SquareBottomDashedScissors aliases
export { default as ScissorsSquareDashedBottom } from '../icons/square-bottom-dashed-scissors.svelte';
// SquareChartGantt aliases
export { default as GanttChartSquare } from '../icons/square-chart-gantt.svelte';
// SquareChartGantt aliases
export { default as SquareGanttChart } from '../icons/square-chart-gantt.svelte';
// SquareCheckBig aliases
export { default as CheckSquare } from '../icons/square-check-big.svelte';
// SquareCheck aliases
export { default as CheckSquare2 } from '../icons/square-check.svelte';
// SquareChevronDown aliases
export { default as ChevronDownSquare } from '../icons/square-chevron-down.svelte';
// SquareChevronLeft aliases
export { default as ChevronLeftSquare } from '../icons/square-chevron-left.svelte';
// SquareChevronRight aliases
export { default as ChevronRightSquare } from '../icons/square-chevron-right.svelte';
// SquareChevronUp aliases
export { default as ChevronUpSquare } from '../icons/square-chevron-up.svelte';
// SquareCode aliases
export { default as CodeSquare } from '../icons/square-code.svelte';
// SquareDashedKanban aliases
export { default as KanbanSquareDashed } from '../icons/square-dashed-kanban.svelte';
// SquareDashedMousePointer aliases
export { default as MousePointerSquareDashed } from '../icons/square-dashed-mouse-pointer.svelte';
// SquareDashed aliases
export { default as BoxSelect } from '../icons/square-dashed.svelte';
// SquareDivide aliases
export { default as DivideSquare } from '../icons/square-divide.svelte';
// SquareDot aliases
export { default as DotSquare } from '../icons/square-dot.svelte';
// SquareEqual aliases
export { default as EqualSquare } from '../icons/square-equal.svelte';
// SquareFunction aliases
export { default as FunctionSquare } from '../icons/square-function.svelte';
// SquareKanban aliases
export { default as KanbanSquare } from '../icons/square-kanban.svelte';
// SquareLibrary aliases
export { default as LibrarySquare } from '../icons/square-library.svelte';
// SquareM aliases
export { default as MSquare } from '../icons/square-m.svelte';
// SquareMenu aliases
export { default as MenuSquare } from '../icons/square-menu.svelte';
// SquareMinus aliases
export { default as MinusSquare } from '../icons/square-minus.svelte';
// SquareMousePointer aliases
export { default as Inspect } from '../icons/square-mouse-pointer.svelte';
// SquareParkingOff aliases
export { default as ParkingSquareOff } from '../icons/square-parking-off.svelte';
// SquareParking aliases
export { default as ParkingSquare } from '../icons/square-parking.svelte';
// SquarePen aliases
export { default as PenBox } from '../icons/square-pen.svelte';
// SquarePen aliases
export { default as Edit } from '../icons/square-pen.svelte';
// SquarePen aliases
export { default as PenSquare } from '../icons/square-pen.svelte';
// SquarePercent aliases
export { default as PercentSquare } from '../icons/square-percent.svelte';
// SquarePi aliases
export { default as PiSquare } from '../icons/square-pi.svelte';
// SquarePilcrow aliases
export { default as PilcrowSquare } from '../icons/square-pilcrow.svelte';
// SquarePlay aliases
export { default as PlaySquare } from '../icons/square-play.svelte';
// SquarePlus aliases
export { default as PlusSquare } from '../icons/square-plus.svelte';
// SquarePower aliases
export { default as PowerSquare } from '../icons/square-power.svelte';
// SquareScissors aliases
export { default as ScissorsSquare } from '../icons/square-scissors.svelte';
// SquareSigma aliases
export { default as SigmaSquare } from '../icons/square-sigma.svelte';
// SquareSlash aliases
export { default as SlashSquare } from '../icons/square-slash.svelte';
// SquareSplitHorizontal aliases
export { default as SplitSquareHorizontal } from '../icons/square-split-horizontal.svelte';
// SquareSplitVertical aliases
export { default as SplitSquareVertical } from '../icons/square-split-vertical.svelte';
// SquareTerminal aliases
export { default as TerminalSquare } from '../icons/square-terminal.svelte';
// SquareUserRound aliases
export { 
/** @deprecated  */
default as UserSquare2 } from '../icons/square-user-round.svelte';
// SquareUser aliases
export { default as UserSquare } from '../icons/square-user.svelte';
// SquareX aliases
export { default as XSquare } from '../icons/square-x.svelte';
// TestTubeDiagonal aliases
export { default as TestTube2 } from '../icons/test-tube-diagonal.svelte';
// TextSelect aliases
export { default as TextSelection } from '../icons/text-select.svelte';
// TramFront aliases
export { default as Train } from '../icons/tram-front.svelte';
// TreePalm aliases
export { default as Palmtree } from '../icons/tree-palm.svelte';
// TriangleAlert aliases
export { default as AlertTriangle } from '../icons/triangle-alert.svelte';
// TvMinimal aliases
export { 
/** @deprecated  */
default as Tv2 } from '../icons/tv-minimal.svelte';
// University aliases
export { 
/** @deprecated  */
default as School2 } from '../icons/university.svelte';
// UserRoundCheck aliases
export { 
/** @deprecated  */
default as UserCheck2 } from '../icons/user-round-check.svelte';
// UserRoundCog aliases
export { 
/** @deprecated  */
default as UserCog2 } from '../icons/user-round-cog.svelte';
// UserRoundMinus aliases
export { 
/** @deprecated  */
default as UserMinus2 } from '../icons/user-round-minus.svelte';
// UserRoundPlus aliases
export { 
/** @deprecated  */
default as UserPlus2 } from '../icons/user-round-plus.svelte';
// UserRoundX aliases
export { 
/** @deprecated  */
default as UserX2 } from '../icons/user-round-x.svelte';
// UserRound aliases
export { 
/** @deprecated  */
default as User2 } from '../icons/user-round.svelte';
// UsersRound aliases
export { 
/** @deprecated  */
default as Users2 } from '../icons/users-round.svelte';
// UtensilsCrossed aliases
export { default as ForkKnifeCrossed } from '../icons/utensils-crossed.svelte';
// Utensils aliases
export { default as ForkKnife } from '../icons/utensils.svelte';
// WalletMinimal aliases
export { 
/** @deprecated  */
default as Wallet2 } from '../icons/wallet-minimal.svelte';
// WandSparkles aliases
export { 
/** @deprecated  */
default as Wand2 } from '../icons/wand-sparkles.svelte';
