// lucide-svelte/dist/aliases/suffixed.js
/**
 * @license lucide-svelte v0.456.0 - ISC
 *
 * This source code is licensed under the ISC license.
 * See the LICENSE file in the root directory of this source tree.
 */
// AArrowDown aliases
export { default as AArrowDownIcon } from '../icons/a-arrow-down.svelte';
// AArrowUp aliases
export { default as AArrowUpIcon } from '../icons/a-arrow-up.svelte';
// ALargeSmall aliases
export { default as ALargeSmallIcon } from '../icons/a-large-small.svelte';
// Accessibility aliases
export { default as AccessibilityIcon } from '../icons/accessibility.svelte';
// Activity aliases
export { default as ActivityIcon } from '../icons/activity.svelte';
// AirVent aliases
export { default as AirVentIcon } from '../icons/air-vent.svelte';
// Airplay aliases
export { default as AirplayIcon } from '../icons/airplay.svelte';
// AlarmClockOff aliases
export { default as AlarmClockOffIcon } from '../icons/alarm-clock-off.svelte';
// AlarmClock aliases
export { default as AlarmClockIcon } from '../icons/alarm-clock.svelte';
// AlarmSmoke aliases
export { default as AlarmSmokeIcon } from '../icons/alarm-smoke.svelte';
// Album aliases
export { default as AlbumIcon } from '../icons/album.svelte';
// AlignCenterHorizontal aliases
export { default as AlignCenterHorizontalIcon } from '../icons/align-center-horizontal.svelte';
// AlignCenterVertical aliases
export { default as AlignCenterVerticalIcon } from '../icons/align-center-vertical.svelte';
// AlignCenter aliases
export { default as AlignCenterIcon } from '../icons/align-center.svelte';
// AlignEndHorizontal aliases
export { default as AlignEndHorizontalIcon } from '../icons/align-end-horizontal.svelte';
// AlignEndVertical aliases
export { default as AlignEndVerticalIcon } from '../icons/align-end-vertical.svelte';
// AlignHorizontalDistributeCenter aliases
export { default as AlignHorizontalDistributeCenterIcon } from '../icons/align-horizontal-distribute-center.svelte';
// AlignHorizontalDistributeEnd aliases
export { default as AlignHorizontalDistributeEndIcon } from '../icons/align-horizontal-distribute-end.svelte';
// AlignHorizontalDistributeStart aliases
export { default as AlignHorizontalDistributeStartIcon } from '../icons/align-horizontal-distribute-start.svelte';
// AlignHorizontalJustifyCenter aliases
export { default as AlignHorizontalJustifyCenterIcon } from '../icons/align-horizontal-justify-center.svelte';
// AlignHorizontalJustifyEnd aliases
export { default as AlignHorizontalJustifyEndIcon } from '../icons/align-horizontal-justify-end.svelte';
// AlignHorizontalJustifyStart aliases
export { default as AlignHorizontalJustifyStartIcon } from '../icons/align-horizontal-justify-start.svelte';
// AlignHorizontalSpaceAround aliases
export { default as AlignHorizontalSpaceAroundIcon } from '../icons/align-horizontal-space-around.svelte';
// AlignHorizontalSpaceBetween aliases
export { default as AlignHorizontalSpaceBetweenIcon } from '../icons/align-horizontal-space-between.svelte';
// AlignJustify aliases
export { default as AlignJustifyIcon } from '../icons/align-justify.svelte';
// AlignLeft aliases
export { default as AlignLeftIcon } from '../icons/align-left.svelte';
// AlignRight aliases
export { default as AlignRightIcon } from '../icons/align-right.svelte';
// AlignStartHorizontal aliases
export { default as AlignStartHorizontalIcon } from '../icons/align-start-horizontal.svelte';
// AlignStartVertical aliases
export { default as AlignStartVerticalIcon } from '../icons/align-start-vertical.svelte';
// AlignVerticalDistributeCenter aliases
export { default as AlignVerticalDistributeCenterIcon } from '../icons/align-vertical-distribute-center.svelte';
// AlignVerticalDistributeEnd aliases
export { default as AlignVerticalDistributeEndIcon } from '../icons/align-vertical-distribute-end.svelte';
// AlignVerticalDistributeStart aliases
export { default as AlignVerticalDistributeStartIcon } from '../icons/align-vertical-distribute-start.svelte';
// AlignVerticalJustifyCenter aliases
export { default as AlignVerticalJustifyCenterIcon } from '../icons/align-vertical-justify-center.svelte';
// AlignVerticalJustifyEnd aliases
export { default as AlignVerticalJustifyEndIcon } from '../icons/align-vertical-justify-end.svelte';
// AlignVerticalJustifyStart aliases
export { default as AlignVerticalJustifyStartIcon } from '../icons/align-vertical-justify-start.svelte';
// AlignVerticalSpaceAround aliases
export { default as AlignVerticalSpaceAroundIcon } from '../icons/align-vertical-space-around.svelte';
// AlignVerticalSpaceBetween aliases
export { default as AlignVerticalSpaceBetweenIcon } from '../icons/align-vertical-space-between.svelte';
// Ambulance aliases
export { default as AmbulanceIcon } from '../icons/ambulance.svelte';
// Ampersand aliases
export { default as AmpersandIcon } from '../icons/ampersand.svelte';
// Ampersands aliases
export { default as AmpersandsIcon } from '../icons/ampersands.svelte';
// Amphora aliases
export { default as AmphoraIcon } from '../icons/amphora.svelte';
// Anchor aliases
export { default as AnchorIcon } from '../icons/anchor.svelte';
// Angry aliases
export { default as AngryIcon } from '../icons/angry.svelte';
// Annoyed aliases
export { default as AnnoyedIcon } from '../icons/annoyed.svelte';
// Antenna aliases
export { default as AntennaIcon } from '../icons/antenna.svelte';
// Anvil aliases
export { default as AnvilIcon } from '../icons/anvil.svelte';
// Aperture aliases
export { default as ApertureIcon } from '../icons/aperture.svelte';
// AppWindowMac aliases
export { default as AppWindowMacIcon } from '../icons/app-window-mac.svelte';
// AppWindow aliases
export { default as AppWindowIcon } from '../icons/app-window.svelte';
// Apple aliases
export { default as AppleIcon } from '../icons/apple.svelte';
// ArchiveRestore aliases
export { default as ArchiveRestoreIcon } from '../icons/archive-restore.svelte';
// ArchiveX aliases
export { default as ArchiveXIcon } from '../icons/archive-x.svelte';
// Archive aliases
export { default as ArchiveIcon } from '../icons/archive.svelte';
// Armchair aliases
export { default as ArmchairIcon } from '../icons/armchair.svelte';
// ArrowBigDownDash aliases
export { default as ArrowBigDownDashIcon } from '../icons/arrow-big-down-dash.svelte';
// ArrowBigDown aliases
export { default as ArrowBigDownIcon } from '../icons/arrow-big-down.svelte';
// ArrowBigLeftDash aliases
export { default as ArrowBigLeftDashIcon } from '../icons/arrow-big-left-dash.svelte';
// ArrowBigLeft aliases
export { default as ArrowBigLeftIcon } from '../icons/arrow-big-left.svelte';
// ArrowBigRightDash aliases
export { default as ArrowBigRightDashIcon } from '../icons/arrow-big-right-dash.svelte';
// ArrowBigRight aliases
export { default as ArrowBigRightIcon } from '../icons/arrow-big-right.svelte';
// ArrowBigUpDash aliases
export { default as ArrowBigUpDashIcon } from '../icons/arrow-big-up-dash.svelte';
// ArrowBigUp aliases
export { default as ArrowBigUpIcon } from '../icons/arrow-big-up.svelte';
// ArrowDownFromLine aliases
export { default as ArrowDownFromLineIcon } from '../icons/arrow-down-from-line.svelte';
// ArrowDownLeft aliases
export { default as ArrowDownLeftIcon } from '../icons/arrow-down-left.svelte';
// ArrowDownNarrowWide aliases
export { default as ArrowDownNarrowWideIcon } from '../icons/arrow-down-narrow-wide.svelte';
// ArrowDownRight aliases
export { default as ArrowDownRightIcon } from '../icons/arrow-down-right.svelte';
// ArrowDownToDot aliases
export { default as ArrowDownToDotIcon } from '../icons/arrow-down-to-dot.svelte';
// ArrowDownToLine aliases
export { default as ArrowDownToLineIcon } from '../icons/arrow-down-to-line.svelte';
// ArrowDownUp aliases
export { default as ArrowDownUpIcon } from '../icons/arrow-down-up.svelte';
// ArrowDown aliases
export { default as ArrowDownIcon } from '../icons/arrow-down.svelte';
// ArrowLeftFromLine aliases
export { default as ArrowLeftFromLineIcon } from '../icons/arrow-left-from-line.svelte';
// ArrowLeftRight aliases
export { default as ArrowLeftRightIcon } from '../icons/arrow-left-right.svelte';
// ArrowLeftToLine aliases
export { default as ArrowLeftToLineIcon } from '../icons/arrow-left-to-line.svelte';
// ArrowLeft aliases
export { default as ArrowLeftIcon } from '../icons/arrow-left.svelte';
// ArrowRightFromLine aliases
export { default as ArrowRightFromLineIcon } from '../icons/arrow-right-from-line.svelte';
// ArrowRightLeft aliases
export { default as ArrowRightLeftIcon } from '../icons/arrow-right-left.svelte';
// ArrowRightToLine aliases
export { default as ArrowRightToLineIcon } from '../icons/arrow-right-to-line.svelte';
// ArrowRight aliases
export { default as ArrowRightIcon } from '../icons/arrow-right.svelte';
// ArrowUpDown aliases
export { default as ArrowUpDownIcon } from '../icons/arrow-up-down.svelte';
// ArrowUpFromDot aliases
export { default as ArrowUpFromDotIcon } from '../icons/arrow-up-from-dot.svelte';
// ArrowUpFromLine aliases
export { default as ArrowUpFromLineIcon } from '../icons/arrow-up-from-line.svelte';
// ArrowUpLeft aliases
export { default as ArrowUpLeftIcon } from '../icons/arrow-up-left.svelte';
// ArrowUpRight aliases
export { default as ArrowUpRightIcon } from '../icons/arrow-up-right.svelte';
// ArrowUpToLine aliases
export { default as ArrowUpToLineIcon } from '../icons/arrow-up-to-line.svelte';
// ArrowUpWideNarrow aliases
export { default as ArrowUpWideNarrowIcon } from '../icons/arrow-up-wide-narrow.svelte';
// ArrowUp aliases
export { default as ArrowUpIcon } from '../icons/arrow-up.svelte';
// ArrowsUpFromLine aliases
export { default as ArrowsUpFromLineIcon } from '../icons/arrows-up-from-line.svelte';
// Asterisk aliases
export { default as AsteriskIcon } from '../icons/asterisk.svelte';
// AtSign aliases
export { default as AtSignIcon } from '../icons/at-sign.svelte';
// Atom aliases
export { default as AtomIcon } from '../icons/atom.svelte';
// AudioLines aliases
export { default as AudioLinesIcon } from '../icons/audio-lines.svelte';
// AudioWaveform aliases
export { default as AudioWaveformIcon } from '../icons/audio-waveform.svelte';
// Award aliases
export { default as AwardIcon } from '../icons/award.svelte';
// Axe aliases
export { default as AxeIcon } from '../icons/axe.svelte';
// Baby aliases
export { default as BabyIcon } from '../icons/baby.svelte';
// Backpack aliases
export { default as BackpackIcon } from '../icons/backpack.svelte';
// BadgeAlert aliases
export { default as BadgeAlertIcon } from '../icons/badge-alert.svelte';
// BadgeCent aliases
export { default as BadgeCentIcon } from '../icons/badge-cent.svelte';
// BadgeDollarSign aliases
export { default as BadgeDollarSignIcon } from '../icons/badge-dollar-sign.svelte';
// BadgeEuro aliases
export { default as BadgeEuroIcon } from '../icons/badge-euro.svelte';
// BadgeHelp aliases
export { default as BadgeHelpIcon } from '../icons/badge-help.svelte';
// BadgeIndianRupee aliases
export { default as BadgeIndianRupeeIcon } from '../icons/badge-indian-rupee.svelte';
// BadgeInfo aliases
export { default as BadgeInfoIcon } from '../icons/badge-info.svelte';
// BadgeJapaneseYen aliases
export { default as BadgeJapaneseYenIcon } from '../icons/badge-japanese-yen.svelte';
// BadgeMinus aliases
export { default as BadgeMinusIcon } from '../icons/badge-minus.svelte';
// BadgePercent aliases
export { default as BadgePercentIcon } from '../icons/badge-percent.svelte';
// BadgePlus aliases
export { default as BadgePlusIcon } from '../icons/badge-plus.svelte';
// BadgePoundSterling aliases
export { default as BadgePoundSterlingIcon } from '../icons/badge-pound-sterling.svelte';
// BadgeRussianRuble aliases
export { default as BadgeRussianRubleIcon } from '../icons/badge-russian-ruble.svelte';
// BadgeSwissFranc aliases
export { default as BadgeSwissFrancIcon } from '../icons/badge-swiss-franc.svelte';
// BadgeX aliases
export { default as BadgeXIcon } from '../icons/badge-x.svelte';
// Badge aliases
export { default as BadgeIcon } from '../icons/badge.svelte';
// BaggageClaim aliases
export { default as BaggageClaimIcon } from '../icons/baggage-claim.svelte';
// Ban aliases
export { default as BanIcon } from '../icons/ban.svelte';
// Banana aliases
export { default as BananaIcon } from '../icons/banana.svelte';
// Bandage aliases
export { default as BandageIcon } from '../icons/bandage.svelte';
// Banknote aliases
export { default as BanknoteIcon } from '../icons/banknote.svelte';
// Barcode aliases
export { default as BarcodeIcon } from '../icons/barcode.svelte';
// Baseline aliases
export { default as BaselineIcon } from '../icons/baseline.svelte';
// Bath aliases
export { default as BathIcon } from '../icons/bath.svelte';
// BatteryCharging aliases
export { default as BatteryChargingIcon } from '../icons/battery-charging.svelte';
// BatteryFull aliases
export { default as BatteryFullIcon } from '../icons/battery-full.svelte';
// BatteryLow aliases
export { default as BatteryLowIcon } from '../icons/battery-low.svelte';
// BatteryMedium aliases
export { default as BatteryMediumIcon } from '../icons/battery-medium.svelte';
// BatteryWarning aliases
export { default as BatteryWarningIcon } from '../icons/battery-warning.svelte';
// Battery aliases
export { default as BatteryIcon } from '../icons/battery.svelte';
// Beaker aliases
export { default as BeakerIcon } from '../icons/beaker.svelte';
// BeanOff aliases
export { default as BeanOffIcon } from '../icons/bean-off.svelte';
// Bean aliases
export { default as BeanIcon } from '../icons/bean.svelte';
// BedDouble aliases
export { default as BedDoubleIcon } from '../icons/bed-double.svelte';
// BedSingle aliases
export { default as BedSingleIcon } from '../icons/bed-single.svelte';
// Bed aliases
export { default as BedIcon } from '../icons/bed.svelte';
// Beef aliases
export { default as BeefIcon } from '../icons/beef.svelte';
// BeerOff aliases
export { default as BeerOffIcon } from '../icons/beer-off.svelte';
// Beer aliases
export { default as BeerIcon } from '../icons/beer.svelte';
// BellDot aliases
export { default as BellDotIcon } from '../icons/bell-dot.svelte';
// BellElectric aliases
export { default as BellElectricIcon } from '../icons/bell-electric.svelte';
// BellMinus aliases
export { default as BellMinusIcon } from '../icons/bell-minus.svelte';
// BellOff aliases
export { default as BellOffIcon } from '../icons/bell-off.svelte';
// BellPlus aliases
export { default as BellPlusIcon } from '../icons/bell-plus.svelte';
// BellRing aliases
export { default as BellRingIcon } from '../icons/bell-ring.svelte';
// Bell aliases
export { default as BellIcon } from '../icons/bell.svelte';
// BetweenVerticalEnd aliases
export { default as BetweenVerticalEndIcon } from '../icons/between-vertical-end.svelte';
// BetweenVerticalStart aliases
export { default as BetweenVerticalStartIcon } from '../icons/between-vertical-start.svelte';
// BicepsFlexed aliases
export { default as BicepsFlexedIcon } from '../icons/biceps-flexed.svelte';
// Bike aliases
export { default as BikeIcon } from '../icons/bike.svelte';
// Binary aliases
export { default as BinaryIcon } from '../icons/binary.svelte';
// Binoculars aliases
export { default as BinocularsIcon } from '../icons/binoculars.svelte';
// Biohazard aliases
export { default as BiohazardIcon } from '../icons/biohazard.svelte';
// Bird aliases
export { default as BirdIcon } from '../icons/bird.svelte';
// Bitcoin aliases
export { default as BitcoinIcon } from '../icons/bitcoin.svelte';
// Blend aliases
export { default as BlendIcon } from '../icons/blend.svelte';
// Blinds aliases
export { default as BlindsIcon } from '../icons/blinds.svelte';
// Blocks aliases
export { default as BlocksIcon } from '../icons/blocks.svelte';
// BluetoothConnected aliases
export { default as BluetoothConnectedIcon } from '../icons/bluetooth-connected.svelte';
// BluetoothOff aliases
export { default as BluetoothOffIcon } from '../icons/bluetooth-off.svelte';
// BluetoothSearching aliases
export { default as BluetoothSearchingIcon } from '../icons/bluetooth-searching.svelte';
// Bluetooth aliases
export { default as BluetoothIcon } from '../icons/bluetooth.svelte';
// Bold aliases
export { default as BoldIcon } from '../icons/bold.svelte';
// Bolt aliases
export { default as BoltIcon } from '../icons/bolt.svelte';
// Bomb aliases
export { default as BombIcon } from '../icons/bomb.svelte';
// Bone aliases
export { default as BoneIcon } from '../icons/bone.svelte';
// BookA aliases
export { default as BookAIcon } from '../icons/book-a.svelte';
// BookAudio aliases
export { default as BookAudioIcon } from '../icons/book-audio.svelte';
// BookCheck aliases
export { default as BookCheckIcon } from '../icons/book-check.svelte';
// BookCopy aliases
export { default as BookCopyIcon } from '../icons/book-copy.svelte';
// BookDown aliases
export { default as BookDownIcon } from '../icons/book-down.svelte';
// BookHeadphones aliases
export { default as BookHeadphonesIcon } from '../icons/book-headphones.svelte';
// BookHeart aliases
export { default as BookHeartIcon } from '../icons/book-heart.svelte';
// BookImage aliases
export { default as BookImageIcon } from '../icons/book-image.svelte';
// BookKey aliases
export { default as BookKeyIcon } from '../icons/book-key.svelte';
// BookLock aliases
export { default as BookLockIcon } from '../icons/book-lock.svelte';
// BookMarked aliases
export { default as BookMarkedIcon } from '../icons/book-marked.svelte';
// BookMinus aliases
export { default as BookMinusIcon } from '../icons/book-minus.svelte';
// BookOpenCheck aliases
export { default as BookOpenCheckIcon } from '../icons/book-open-check.svelte';
// BookOpenText aliases
export { default as BookOpenTextIcon } from '../icons/book-open-text.svelte';
// BookOpen aliases
export { default as BookOpenIcon } from '../icons/book-open.svelte';
// BookPlus aliases
export { default as BookPlusIcon } from '../icons/book-plus.svelte';
// BookText aliases
export { default as BookTextIcon } from '../icons/book-text.svelte';
// BookType aliases
export { default as BookTypeIcon } from '../icons/book-type.svelte';
// BookUp2 aliases
export { default as BookUp2Icon } from '../icons/book-up-2.svelte';
// BookUp aliases
export { default as BookUpIcon } from '../icons/book-up.svelte';
// BookUser aliases
export { default as BookUserIcon } from '../icons/book-user.svelte';
// BookX aliases
export { default as BookXIcon } from '../icons/book-x.svelte';
// Book aliases
export { default as BookIcon } from '../icons/book.svelte';
// BookmarkCheck aliases
export { default as BookmarkCheckIcon } from '../icons/bookmark-check.svelte';
// BookmarkMinus aliases
export { default as BookmarkMinusIcon } from '../icons/bookmark-minus.svelte';
// BookmarkPlus aliases
export { default as BookmarkPlusIcon } from '../icons/bookmark-plus.svelte';
// BookmarkX aliases
export { default as BookmarkXIcon } from '../icons/bookmark-x.svelte';
// Bookmark aliases
export { default as BookmarkIcon } from '../icons/bookmark.svelte';
// BoomBox aliases
export { default as BoomBoxIcon } from '../icons/boom-box.svelte';
// BotMessageSquare aliases
export { default as BotMessageSquareIcon } from '../icons/bot-message-square.svelte';
// BotOff aliases
export { default as BotOffIcon } from '../icons/bot-off.svelte';
// Bot aliases
export { default as BotIcon } from '../icons/bot.svelte';
// Box aliases
export { default as BoxIcon } from '../icons/box.svelte';
// Boxes aliases
export { default as BoxesIcon } from '../icons/boxes.svelte';
// Brackets aliases
export { default as BracketsIcon } from '../icons/brackets.svelte';
// BrainCircuit aliases
export { default as BrainCircuitIcon } from '../icons/brain-circuit.svelte';
// BrainCog aliases
export { default as BrainCogIcon } from '../icons/brain-cog.svelte';
// Brain aliases
export { default as BrainIcon } from '../icons/brain.svelte';
// BrickWall aliases
export { default as BrickWallIcon } from '../icons/brick-wall.svelte';
// BriefcaseBusiness aliases
export { default as BriefcaseBusinessIcon } from '../icons/briefcase-business.svelte';
// BriefcaseConveyorBelt aliases
export { default as BriefcaseConveyorBeltIcon } from '../icons/briefcase-conveyor-belt.svelte';
// BriefcaseMedical aliases
export { default as BriefcaseMedicalIcon } from '../icons/briefcase-medical.svelte';
// Briefcase aliases
export { default as BriefcaseIcon } from '../icons/briefcase.svelte';
// BringToFront aliases
export { default as BringToFrontIcon } from '../icons/bring-to-front.svelte';
// Brush aliases
export { default as BrushIcon } from '../icons/brush.svelte';
// BugOff aliases
export { default as BugOffIcon } from '../icons/bug-off.svelte';
// BugPlay aliases
export { default as BugPlayIcon } from '../icons/bug-play.svelte';
// Bug aliases
export { default as BugIcon } from '../icons/bug.svelte';
// Building2 aliases
export { default as Building2Icon } from '../icons/building-2.svelte';
// Building aliases
export { default as BuildingIcon } from '../icons/building.svelte';
// BusFront aliases
export { default as BusFrontIcon } from '../icons/bus-front.svelte';
// Bus aliases
export { default as BusIcon } from '../icons/bus.svelte';
// CableCar aliases
export { default as CableCarIcon } from '../icons/cable-car.svelte';
// Cable aliases
export { default as CableIcon } from '../icons/cable.svelte';
// CakeSlice aliases
export { default as CakeSliceIcon } from '../icons/cake-slice.svelte';
// Cake aliases
export { default as CakeIcon } from '../icons/cake.svelte';
// Calculator aliases
export { default as CalculatorIcon } from '../icons/calculator.svelte';
// Calendar1 aliases
export { default as Calendar1Icon } from '../icons/calendar-1.svelte';
// CalendarArrowDown aliases
export { default as CalendarArrowDownIcon } from '../icons/calendar-arrow-down.svelte';
// CalendarArrowUp aliases
export { default as CalendarArrowUpIcon } from '../icons/calendar-arrow-up.svelte';
// CalendarCheck2 aliases
export { default as CalendarCheck2Icon } from '../icons/calendar-check-2.svelte';
// CalendarCheck aliases
export { default as CalendarCheckIcon } from '../icons/calendar-check.svelte';
// CalendarClock aliases
export { default as CalendarClockIcon } from '../icons/calendar-clock.svelte';
// CalendarCog aliases
export { default as CalendarCogIcon } from '../icons/calendar-cog.svelte';
// CalendarDays aliases
export { default as CalendarDaysIcon } from '../icons/calendar-days.svelte';
// CalendarFold aliases
export { default as CalendarFoldIcon } from '../icons/calendar-fold.svelte';
// CalendarHeart aliases
export { default as CalendarHeartIcon } from '../icons/calendar-heart.svelte';
// CalendarMinus2 aliases
export { default as CalendarMinus2Icon } from '../icons/calendar-minus-2.svelte';
// CalendarMinus aliases
export { default as CalendarMinusIcon } from '../icons/calendar-minus.svelte';
// CalendarOff aliases
export { default as CalendarOffIcon } from '../icons/calendar-off.svelte';
// CalendarPlus2 aliases
export { default as CalendarPlus2Icon } from '../icons/calendar-plus-2.svelte';
// CalendarPlus aliases
export { default as CalendarPlusIcon } from '../icons/calendar-plus.svelte';
// CalendarRange aliases
export { default as CalendarRangeIcon } from '../icons/calendar-range.svelte';
// CalendarSearch aliases
export { default as CalendarSearchIcon } from '../icons/calendar-search.svelte';
// CalendarX2 aliases
export { default as CalendarX2Icon } from '../icons/calendar-x-2.svelte';
// CalendarX aliases
export { default as CalendarXIcon } from '../icons/calendar-x.svelte';
// Calendar aliases
export { default as CalendarIcon } from '../icons/calendar.svelte';
// CameraOff aliases
export { default as CameraOffIcon } from '../icons/camera-off.svelte';
// Camera aliases
export { default as CameraIcon } from '../icons/camera.svelte';
// CandyCane aliases
export { default as CandyCaneIcon } from '../icons/candy-cane.svelte';
// CandyOff aliases
export { default as CandyOffIcon } from '../icons/candy-off.svelte';
// Candy aliases
export { default as CandyIcon } from '../icons/candy.svelte';
// Cannabis aliases
export { default as CannabisIcon } from '../icons/cannabis.svelte';
// CaptionsOff aliases
export { default as CaptionsOffIcon } from '../icons/captions-off.svelte';
// CarFront aliases
export { default as CarFrontIcon } from '../icons/car-front.svelte';
// CarTaxiFront aliases
export { default as CarTaxiFrontIcon } from '../icons/car-taxi-front.svelte';
// Car aliases
export { default as CarIcon } from '../icons/car.svelte';
// Caravan aliases
export { default as CaravanIcon } from '../icons/caravan.svelte';
// Carrot aliases
export { default as CarrotIcon } from '../icons/carrot.svelte';
// CaseLower aliases
export { default as CaseLowerIcon } from '../icons/case-lower.svelte';
// CaseSensitive aliases
export { default as CaseSensitiveIcon } from '../icons/case-sensitive.svelte';
// CaseUpper aliases
export { default as CaseUpperIcon } from '../icons/case-upper.svelte';
// CassetteTape aliases
export { default as CassetteTapeIcon } from '../icons/cassette-tape.svelte';
// Cast aliases
export { default as CastIcon } from '../icons/cast.svelte';
// Castle aliases
export { default as CastleIcon } from '../icons/castle.svelte';
// Cat aliases
export { default as CatIcon } from '../icons/cat.svelte';
// Cctv aliases
export { default as CctvIcon } from '../icons/cctv.svelte';
// ChartBarDecreasing aliases
export { default as ChartBarDecreasingIcon } from '../icons/chart-bar-decreasing.svelte';
// ChartBarIncreasing aliases
export { default as ChartBarIncreasingIcon } from '../icons/chart-bar-increasing.svelte';
// ChartBarStacked aliases
export { default as ChartBarStackedIcon } from '../icons/chart-bar-stacked.svelte';
// ChartColumnDecreasing aliases
export { default as ChartColumnDecreasingIcon } from '../icons/chart-column-decreasing.svelte';
// ChartColumnStacked aliases
export { default as ChartColumnStackedIcon } from '../icons/chart-column-stacked.svelte';
// ChartGantt aliases
export { default as ChartGanttIcon } from '../icons/chart-gantt.svelte';
// ChartNetwork aliases
export { default as ChartNetworkIcon } from '../icons/chart-network.svelte';
// ChartNoAxesColumnDecreasing aliases
export { default as ChartNoAxesColumnDecreasingIcon } from '../icons/chart-no-axes-column-decreasing.svelte';
// ChartNoAxesCombined aliases
export { default as ChartNoAxesCombinedIcon } from '../icons/chart-no-axes-combined.svelte';
// ChartSpline aliases
export { default as ChartSplineIcon } from '../icons/chart-spline.svelte';
// CheckCheck aliases
export { default as CheckCheckIcon } from '../icons/check-check.svelte';
// Check aliases
export { default as CheckIcon } from '../icons/check.svelte';
// ChefHat aliases
export { default as ChefHatIcon } from '../icons/chef-hat.svelte';
// Cherry aliases
export { default as CherryIcon } from '../icons/cherry.svelte';
// ChevronDown aliases
export { default as ChevronDownIcon } from '../icons/chevron-down.svelte';
// ChevronFirst aliases
export { default as ChevronFirstIcon } from '../icons/chevron-first.svelte';
// ChevronLast aliases
export { default as ChevronLastIcon } from '../icons/chevron-last.svelte';
// ChevronLeft aliases
export { default as ChevronLeftIcon } from '../icons/chevron-left.svelte';
// ChevronRight aliases
export { default as ChevronRightIcon } from '../icons/chevron-right.svelte';
// ChevronUp aliases
export { default as ChevronUpIcon } from '../icons/chevron-up.svelte';
// ChevronsDownUp aliases
export { default as ChevronsDownUpIcon } from '../icons/chevrons-down-up.svelte';
// ChevronsDown aliases
export { default as ChevronsDownIcon } from '../icons/chevrons-down.svelte';
// ChevronsLeftRightEllipsis aliases
export { default as ChevronsLeftRightEllipsisIcon } from '../icons/chevrons-left-right-ellipsis.svelte';
// ChevronsLeftRight aliases
export { default as ChevronsLeftRightIcon } from '../icons/chevrons-left-right.svelte';
// ChevronsLeft aliases
export { default as ChevronsLeftIcon } from '../icons/chevrons-left.svelte';
// ChevronsRightLeft aliases
export { default as ChevronsRightLeftIcon } from '../icons/chevrons-right-left.svelte';
// ChevronsRight aliases
export { default as ChevronsRightIcon } from '../icons/chevrons-right.svelte';
// ChevronsUpDown aliases
export { default as ChevronsUpDownIcon } from '../icons/chevrons-up-down.svelte';
// ChevronsUp aliases
export { default as ChevronsUpIcon } from '../icons/chevrons-up.svelte';
// Chrome aliases
export { default as ChromeIcon } from '../icons/chrome.svelte';
// Church aliases
export { default as ChurchIcon } from '../icons/church.svelte';
// CigaretteOff aliases
export { default as CigaretteOffIcon } from '../icons/cigarette-off.svelte';
// Cigarette aliases
export { default as CigaretteIcon } from '../icons/cigarette.svelte';
// CircleDashed aliases
export { default as CircleDashedIcon } from '../icons/circle-dashed.svelte';
// CircleDollarSign aliases
export { default as CircleDollarSignIcon } from '../icons/circle-dollar-sign.svelte';
// CircleDotDashed aliases
export { default as CircleDotDashedIcon } from '../icons/circle-dot-dashed.svelte';
// CircleDot aliases
export { default as CircleDotIcon } from '../icons/circle-dot.svelte';
// CircleEllipsis aliases
export { default as CircleEllipsisIcon } from '../icons/circle-ellipsis.svelte';
// CircleEqual aliases
export { default as CircleEqualIcon } from '../icons/circle-equal.svelte';
// CircleFadingArrowUp aliases
export { default as CircleFadingArrowUpIcon } from '../icons/circle-fading-arrow-up.svelte';
// CircleFadingPlus aliases
export { default as CircleFadingPlusIcon } from '../icons/circle-fading-plus.svelte';
// CircleOff aliases
export { default as CircleOffIcon } from '../icons/circle-off.svelte';
// CircleSlash aliases
export { default as CircleSlashIcon } from '../icons/circle-slash.svelte';
// Circle aliases
export { default as CircleIcon } from '../icons/circle.svelte';
// CircuitBoard aliases
export { default as CircuitBoardIcon } from '../icons/circuit-board.svelte';
// Citrus aliases
export { default as CitrusIcon } from '../icons/citrus.svelte';
// Clapperboard aliases
export { default as ClapperboardIcon } from '../icons/clapperboard.svelte';
// ClipboardCheck aliases
export { default as ClipboardCheckIcon } from '../icons/clipboard-check.svelte';
// ClipboardCopy aliases
export { default as ClipboardCopyIcon } from '../icons/clipboard-copy.svelte';
// ClipboardList aliases
export { default as ClipboardListIcon } from '../icons/clipboard-list.svelte';
// ClipboardMinus aliases
export { default as ClipboardMinusIcon } from '../icons/clipboard-minus.svelte';
// ClipboardPaste aliases
export { default as ClipboardPasteIcon } from '../icons/clipboard-paste.svelte';
// ClipboardPlus aliases
export { default as ClipboardPlusIcon } from '../icons/clipboard-plus.svelte';
// ClipboardType aliases
export { default as ClipboardTypeIcon } from '../icons/clipboard-type.svelte';
// ClipboardX aliases
export { default as ClipboardXIcon } from '../icons/clipboard-x.svelte';
// Clipboard aliases
export { default as ClipboardIcon } from '../icons/clipboard.svelte';
// Clock1 aliases
export { default as Clock1Icon } from '../icons/clock-1.svelte';
// Clock10 aliases
export { default as Clock10Icon } from '../icons/clock-10.svelte';
// Clock11 aliases
export { default as Clock11Icon } from '../icons/clock-11.svelte';
// Clock12 aliases
export { default as Clock12Icon } from '../icons/clock-12.svelte';
// Clock2 aliases
export { default as Clock2Icon } from '../icons/clock-2.svelte';
// Clock3 aliases
export { default as Clock3Icon } from '../icons/clock-3.svelte';
// Clock4 aliases
export { default as Clock4Icon } from '../icons/clock-4.svelte';
// Clock5 aliases
export { default as Clock5Icon } from '../icons/clock-5.svelte';
// Clock6 aliases
export { default as Clock6Icon } from '../icons/clock-6.svelte';
// Clock7 aliases
export { default as Clock7Icon } from '../icons/clock-7.svelte';
// Clock8 aliases
export { default as Clock8Icon } from '../icons/clock-8.svelte';
// Clock9 aliases
export { default as Clock9Icon } from '../icons/clock-9.svelte';
// ClockAlert aliases
export { default as ClockAlertIcon } from '../icons/clock-alert.svelte';
// ClockArrowDown aliases
export { default as ClockArrowDownIcon } from '../icons/clock-arrow-down.svelte';
// ClockArrowUp aliases
export { default as ClockArrowUpIcon } from '../icons/clock-arrow-up.svelte';
// Clock aliases
export { default as ClockIcon } from '../icons/clock.svelte';
// CloudCog aliases
export { default as CloudCogIcon } from '../icons/cloud-cog.svelte';
// CloudDrizzle aliases
export { default as CloudDrizzleIcon } from '../icons/cloud-drizzle.svelte';
// CloudFog aliases
export { default as CloudFogIcon } from '../icons/cloud-fog.svelte';
// CloudHail aliases
export { default as CloudHailIcon } from '../icons/cloud-hail.svelte';
// CloudLightning aliases
export { default as CloudLightningIcon } from '../icons/cloud-lightning.svelte';
// CloudMoonRain aliases
export { default as CloudMoonRainIcon } from '../icons/cloud-moon-rain.svelte';
// CloudMoon aliases
export { default as CloudMoonIcon } from '../icons/cloud-moon.svelte';
// CloudOff aliases
export { default as CloudOffIcon } from '../icons/cloud-off.svelte';
// CloudRainWind aliases
export { default as CloudRainWindIcon } from '../icons/cloud-rain-wind.svelte';
// CloudRain aliases
export { default as CloudRainIcon } from '../icons/cloud-rain.svelte';
// CloudSnow aliases
export { default as CloudSnowIcon } from '../icons/cloud-snow.svelte';
// CloudSunRain aliases
export { default as CloudSunRainIcon } from '../icons/cloud-sun-rain.svelte';
// CloudSun aliases
export { default as CloudSunIcon } from '../icons/cloud-sun.svelte';
// Cloud aliases
export { default as CloudIcon } from '../icons/cloud.svelte';
// Cloudy aliases
export { default as CloudyIcon } from '../icons/cloudy.svelte';
// Clover aliases
export { default as CloverIcon } from '../icons/clover.svelte';
// Club aliases
export { default as ClubIcon } from '../icons/club.svelte';
// Code aliases
export { default as CodeIcon } from '../icons/code.svelte';
// Codepen aliases
export { default as CodepenIcon } from '../icons/codepen.svelte';
// Codesandbox aliases
export { default as CodesandboxIcon } from '../icons/codesandbox.svelte';
// Coffee aliases
export { default as CoffeeIcon } from '../icons/coffee.svelte';
// Cog aliases
export { default as CogIcon } from '../icons/cog.svelte';
// Coins aliases
export { default as CoinsIcon } from '../icons/coins.svelte';
// Columns4 aliases
export { default as Columns4Icon } from '../icons/columns-4.svelte';
// Combine aliases
export { default as CombineIcon } from '../icons/combine.svelte';
// Command aliases
export { default as CommandIcon } from '../icons/command.svelte';
// Compass aliases
export { default as CompassIcon } from '../icons/compass.svelte';
// Component aliases
export { default as ComponentIcon } from '../icons/component.svelte';
// Computer aliases
export { default as ComputerIcon } from '../icons/computer.svelte';
// ConciergeBell aliases
export { default as ConciergeBellIcon } from '../icons/concierge-bell.svelte';
// Cone aliases
export { default as ConeIcon } from '../icons/cone.svelte';
// Construction aliases
export { default as ConstructionIcon } from '../icons/construction.svelte';
// Contact aliases
export { default as ContactIcon } from '../icons/contact.svelte';
// Container aliases
export { default as ContainerIcon } from '../icons/container.svelte';
// Contrast aliases
export { default as ContrastIcon } from '../icons/contrast.svelte';
// Cookie aliases
export { default as CookieIcon } from '../icons/cookie.svelte';
// CookingPot aliases
export { default as CookingPotIcon } from '../icons/cooking-pot.svelte';
// CopyCheck aliases
export { default as CopyCheckIcon } from '../icons/copy-check.svelte';
// CopyMinus aliases
export { default as CopyMinusIcon } from '../icons/copy-minus.svelte';
// CopyPlus aliases
export { default as CopyPlusIcon } from '../icons/copy-plus.svelte';
// CopySlash aliases
export { default as CopySlashIcon } from '../icons/copy-slash.svelte';
// CopyX aliases
export { default as CopyXIcon } from '../icons/copy-x.svelte';
// Copy aliases
export { default as CopyIcon } from '../icons/copy.svelte';
// Copyleft aliases
export { default as CopyleftIcon } from '../icons/copyleft.svelte';
// Copyright aliases
export { default as CopyrightIcon } from '../icons/copyright.svelte';
// CornerDownLeft aliases
export { default as CornerDownLeftIcon } from '../icons/corner-down-left.svelte';
// CornerDownRight aliases
export { default as CornerDownRightIcon } from '../icons/corner-down-right.svelte';
// CornerLeftDown aliases
export { default as CornerLeftDownIcon } from '../icons/corner-left-down.svelte';
// CornerLeftUp aliases
export { default as CornerLeftUpIcon } from '../icons/corner-left-up.svelte';
// CornerRightDown aliases
export { default as CornerRightDownIcon } from '../icons/corner-right-down.svelte';
// CornerRightUp aliases
export { default as CornerRightUpIcon } from '../icons/corner-right-up.svelte';
// CornerUpLeft aliases
export { default as CornerUpLeftIcon } from '../icons/corner-up-left.svelte';
// CornerUpRight aliases
export { default as CornerUpRightIcon } from '../icons/corner-up-right.svelte';
// Cpu aliases
export { default as CpuIcon } from '../icons/cpu.svelte';
// CreativeCommons aliases
export { default as CreativeCommonsIcon } from '../icons/creative-commons.svelte';
// CreditCard aliases
export { default as CreditCardIcon } from '../icons/credit-card.svelte';
// Croissant aliases
export { default as CroissantIcon } from '../icons/croissant.svelte';
// Crop aliases
export { default as CropIcon } from '../icons/crop.svelte';
// Cross aliases
export { default as CrossIcon } from '../icons/cross.svelte';
// Crosshair aliases
export { default as CrosshairIcon } from '../icons/crosshair.svelte';
// Crown aliases
export { default as CrownIcon } from '../icons/crown.svelte';
// Cuboid aliases
export { default as CuboidIcon } from '../icons/cuboid.svelte';
// CupSoda aliases
export { default as CupSodaIcon } from '../icons/cup-soda.svelte';
// Currency aliases
export { default as CurrencyIcon } from '../icons/currency.svelte';
// Cylinder aliases
export { default as CylinderIcon } from '../icons/cylinder.svelte';
// Dam aliases
export { default as DamIcon } from '../icons/dam.svelte';
// DatabaseBackup aliases
export { default as DatabaseBackupIcon } from '../icons/database-backup.svelte';
// DatabaseZap aliases
export { default as DatabaseZapIcon } from '../icons/database-zap.svelte';
// Database aliases
export { default as DatabaseIcon } from '../icons/database.svelte';
// Delete aliases
export { default as DeleteIcon } from '../icons/delete.svelte';
// Dessert aliases
export { default as DessertIcon } from '../icons/dessert.svelte';
// Diameter aliases
export { default as DiameterIcon } from '../icons/diameter.svelte';
// DiamondMinus aliases
export { default as DiamondMinusIcon } from '../icons/diamond-minus.svelte';
// DiamondPlus aliases
export { default as DiamondPlusIcon } from '../icons/diamond-plus.svelte';
// Diamond aliases
export { default as DiamondIcon } from '../icons/diamond.svelte';
// Dice1 aliases
export { default as Dice1Icon } from '../icons/dice-1.svelte';
// Dice2 aliases
export { default as Dice2Icon } from '../icons/dice-2.svelte';
// Dice3 aliases
export { default as Dice3Icon } from '../icons/dice-3.svelte';
// Dice4 aliases
export { default as Dice4Icon } from '../icons/dice-4.svelte';
// Dice5 aliases
export { default as Dice5Icon } from '../icons/dice-5.svelte';
// Dice6 aliases
export { default as Dice6Icon } from '../icons/dice-6.svelte';
// Dices aliases
export { default as DicesIcon } from '../icons/dices.svelte';
// Diff aliases
export { default as DiffIcon } from '../icons/diff.svelte';
// Disc2 aliases
export { default as Disc2Icon } from '../icons/disc-2.svelte';
// Disc3 aliases
export { default as Disc3Icon } from '../icons/disc-3.svelte';
// DiscAlbum aliases
export { default as DiscAlbumIcon } from '../icons/disc-album.svelte';
// Disc aliases
export { default as DiscIcon } from '../icons/disc.svelte';
// Divide aliases
export { default as DivideIcon } from '../icons/divide.svelte';
// DnaOff aliases
export { default as DnaOffIcon } from '../icons/dna-off.svelte';
// Dna aliases
export { default as DnaIcon } from '../icons/dna.svelte';
// Dock aliases
export { default as DockIcon } from '../icons/dock.svelte';
// Dog aliases
export { default as DogIcon } from '../icons/dog.svelte';
// DollarSign aliases
export { default as DollarSignIcon } from '../icons/dollar-sign.svelte';
// Donut aliases
export { default as DonutIcon } from '../icons/donut.svelte';
// DoorClosed aliases
export { default as DoorClosedIcon } from '../icons/door-closed.svelte';
// DoorOpen aliases
export { default as DoorOpenIcon } from '../icons/door-open.svelte';
// Dot aliases
export { default as DotIcon } from '../icons/dot.svelte';
// Download aliases
export { default as DownloadIcon } from '../icons/download.svelte';
// DraftingCompass aliases
export { default as DraftingCompassIcon } from '../icons/drafting-compass.svelte';
// Drama aliases
export { default as DramaIcon } from '../icons/drama.svelte';
// Dribbble aliases
export { default as DribbbleIcon } from '../icons/dribbble.svelte';
// Drill aliases
export { default as DrillIcon } from '../icons/drill.svelte';
// Droplet aliases
export { default as DropletIcon } from '../icons/droplet.svelte';
// Droplets aliases
export { default as DropletsIcon } from '../icons/droplets.svelte';
// Drum aliases
export { default as DrumIcon } from '../icons/drum.svelte';
// Drumstick aliases
export { default as DrumstickIcon } from '../icons/drumstick.svelte';
// Dumbbell aliases
export { default as DumbbellIcon } from '../icons/dumbbell.svelte';
// EarOff aliases
export { default as EarOffIcon } from '../icons/ear-off.svelte';
// Ear aliases
export { default as EarIcon } from '../icons/ear.svelte';
// EarthLock aliases
export { default as EarthLockIcon } from '../icons/earth-lock.svelte';
// Eclipse aliases
export { default as EclipseIcon } from '../icons/eclipse.svelte';
// EggFried aliases
export { default as EggFriedIcon } from '../icons/egg-fried.svelte';
// EggOff aliases
export { default as EggOffIcon } from '../icons/egg-off.svelte';
// Egg aliases
export { default as EggIcon } from '../icons/egg.svelte';
// EqualNot aliases
export { default as EqualNotIcon } from '../icons/equal-not.svelte';
// Equal aliases
export { default as EqualIcon } from '../icons/equal.svelte';
// Eraser aliases
export { default as EraserIcon } from '../icons/eraser.svelte';
// EthernetPort aliases
export { default as EthernetPortIcon } from '../icons/ethernet-port.svelte';
// Euro aliases
export { default as EuroIcon } from '../icons/euro.svelte';
// Expand aliases
export { default as ExpandIcon } from '../icons/expand.svelte';
// ExternalLink aliases
export { default as ExternalLinkIcon } from '../icons/external-link.svelte';
// EyeClosed aliases
export { default as EyeClosedIcon } from '../icons/eye-closed.svelte';
// EyeOff aliases
export { default as EyeOffIcon } from '../icons/eye-off.svelte';
// Eye aliases
export { default as EyeIcon } from '../icons/eye.svelte';
// Facebook aliases
export { default as FacebookIcon } from '../icons/facebook.svelte';
// Factory aliases
export { default as FactoryIcon } from '../icons/factory.svelte';
// Fan aliases
export { default as FanIcon } from '../icons/fan.svelte';
// FastForward aliases
export { default as FastForwardIcon } from '../icons/fast-forward.svelte';
// Feather aliases
export { default as FeatherIcon } from '../icons/feather.svelte';
// Fence aliases
export { default as FenceIcon } from '../icons/fence.svelte';
// FerrisWheel aliases
export { default as FerrisWheelIcon } from '../icons/ferris-wheel.svelte';
// Figma aliases
export { default as FigmaIcon } from '../icons/figma.svelte';
// FileArchive aliases
export { default as FileArchiveIcon } from '../icons/file-archive.svelte';
// FileAudio2 aliases
export { default as FileAudio2Icon } from '../icons/file-audio-2.svelte';
// FileAudio aliases
export { default as FileAudioIcon } from '../icons/file-audio.svelte';
// FileBadge2 aliases
export { default as FileBadge2Icon } from '../icons/file-badge-2.svelte';
// FileBadge aliases
export { default as FileBadgeIcon } from '../icons/file-badge.svelte';
// FileBox aliases
export { default as FileBoxIcon } from '../icons/file-box.svelte';
// FileCheck2 aliases
export { default as FileCheck2Icon } from '../icons/file-check-2.svelte';
// FileCheck aliases
export { default as FileCheckIcon } from '../icons/file-check.svelte';
// FileClock aliases
export { default as FileClockIcon } from '../icons/file-clock.svelte';
// FileCode2 aliases
export { default as FileCode2Icon } from '../icons/file-code-2.svelte';
// FileCode aliases
export { default as FileCodeIcon } from '../icons/file-code.svelte';
// FileDiff aliases
export { default as FileDiffIcon } from '../icons/file-diff.svelte';
// FileDigit aliases
export { default as FileDigitIcon } from '../icons/file-digit.svelte';
// FileDown aliases
export { default as FileDownIcon } from '../icons/file-down.svelte';
// FileHeart aliases
export { default as FileHeartIcon } from '../icons/file-heart.svelte';
// FileImage aliases
export { default as FileImageIcon } from '../icons/file-image.svelte';
// FileInput aliases
export { default as FileInputIcon } from '../icons/file-input.svelte';
// FileJson2 aliases
export { default as FileJson2Icon } from '../icons/file-json-2.svelte';
// FileJson aliases
export { default as FileJsonIcon } from '../icons/file-json.svelte';
// FileKey2 aliases
export { default as FileKey2Icon } from '../icons/file-key-2.svelte';
// FileKey aliases
export { default as FileKeyIcon } from '../icons/file-key.svelte';
// FileLock2 aliases
export { default as FileLock2Icon } from '../icons/file-lock-2.svelte';
// FileLock aliases
export { default as FileLockIcon } from '../icons/file-lock.svelte';
// FileMinus2 aliases
export { default as FileMinus2Icon } from '../icons/file-minus-2.svelte';
// FileMinus aliases
export { default as FileMinusIcon } from '../icons/file-minus.svelte';
// FileMusic aliases
export { default as FileMusicIcon } from '../icons/file-music.svelte';
// FileOutput aliases
export { default as FileOutputIcon } from '../icons/file-output.svelte';
// FilePlus2 aliases
export { default as FilePlus2Icon } from '../icons/file-plus-2.svelte';
// FilePlus aliases
export { default as FilePlusIcon } from '../icons/file-plus.svelte';
// FileQuestion aliases
export { default as FileQuestionIcon } from '../icons/file-question.svelte';
// FileScan aliases
export { default as FileScanIcon } from '../icons/file-scan.svelte';
// FileSearch2 aliases
export { default as FileSearch2Icon } from '../icons/file-search-2.svelte';
// FileSearch aliases
export { default as FileSearchIcon } from '../icons/file-search.svelte';
// FileSliders aliases
export { default as FileSlidersIcon } from '../icons/file-sliders.svelte';
// FileSpreadsheet aliases
export { default as FileSpreadsheetIcon } from '../icons/file-spreadsheet.svelte';
// FileStack aliases
export { default as FileStackIcon } from '../icons/file-stack.svelte';
// FileSymlink aliases
export { default as FileSymlinkIcon } from '../icons/file-symlink.svelte';
// FileTerminal aliases
export { default as FileTerminalIcon } from '../icons/file-terminal.svelte';
// FileText aliases
export { default as FileTextIcon } from '../icons/file-text.svelte';
// FileType2 aliases
export { default as FileType2Icon } from '../icons/file-type-2.svelte';
// FileType aliases
export { default as FileTypeIcon } from '../icons/file-type.svelte';
// FileUp aliases
export { default as FileUpIcon } from '../icons/file-up.svelte';
// FileUser aliases
export { default as FileUserIcon } from '../icons/file-user.svelte';
// FileVideo2 aliases
export { default as FileVideo2Icon } from '../icons/file-video-2.svelte';
// FileVideo aliases
export { default as FileVideoIcon } from '../icons/file-video.svelte';
// FileVolume2 aliases
export { default as FileVolume2Icon } from '../icons/file-volume-2.svelte';
// FileVolume aliases
export { default as FileVolumeIcon } from '../icons/file-volume.svelte';
// FileWarning aliases
export { default as FileWarningIcon } from '../icons/file-warning.svelte';
// FileX2 aliases
export { default as FileX2Icon } from '../icons/file-x-2.svelte';
// FileX aliases
export { default as FileXIcon } from '../icons/file-x.svelte';
// File aliases
export { default as FileIcon } from '../icons/file.svelte';
// Files aliases
export { default as FilesIcon } from '../icons/files.svelte';
// Film aliases
export { default as FilmIcon } from '../icons/film.svelte';
// FilterX aliases
export { default as FilterXIcon } from '../icons/filter-x.svelte';
// Filter aliases
export { default as FilterIcon } from '../icons/filter.svelte';
// Fingerprint aliases
export { default as FingerprintIcon } from '../icons/fingerprint.svelte';
// FireExtinguisher aliases
export { default as FireExtinguisherIcon } from '../icons/fire-extinguisher.svelte';
// FishOff aliases
export { default as FishOffIcon } from '../icons/fish-off.svelte';
// FishSymbol aliases
export { default as FishSymbolIcon } from '../icons/fish-symbol.svelte';
// Fish aliases
export { default as FishIcon } from '../icons/fish.svelte';
// FlagOff aliases
export { default as FlagOffIcon } from '../icons/flag-off.svelte';
// FlagTriangleLeft aliases
export { default as FlagTriangleLeftIcon } from '../icons/flag-triangle-left.svelte';
// FlagTriangleRight aliases
export { default as FlagTriangleRightIcon } from '../icons/flag-triangle-right.svelte';
// Flag aliases
export { default as FlagIcon } from '../icons/flag.svelte';
// FlameKindling aliases
export { default as FlameKindlingIcon } from '../icons/flame-kindling.svelte';
// Flame aliases
export { default as FlameIcon } from '../icons/flame.svelte';
// FlashlightOff aliases
export { default as FlashlightOffIcon } from '../icons/flashlight-off.svelte';
// Flashlight aliases
export { default as FlashlightIcon } from '../icons/flashlight.svelte';
// FlaskConicalOff aliases
export { default as FlaskConicalOffIcon } from '../icons/flask-conical-off.svelte';
// FlaskConical aliases
export { default as FlaskConicalIcon } from '../icons/flask-conical.svelte';
// FlaskRound aliases
export { default as FlaskRoundIcon } from '../icons/flask-round.svelte';
// FlipHorizontal2 aliases
export { default as FlipHorizontal2Icon } from '../icons/flip-horizontal-2.svelte';
// FlipHorizontal aliases
export { default as FlipHorizontalIcon } from '../icons/flip-horizontal.svelte';
// FlipVertical2 aliases
export { default as FlipVertical2Icon } from '../icons/flip-vertical-2.svelte';
// FlipVertical aliases
export { default as FlipVerticalIcon } from '../icons/flip-vertical.svelte';
// Flower2 aliases
export { default as Flower2Icon } from '../icons/flower-2.svelte';
// Flower aliases
export { default as FlowerIcon } from '../icons/flower.svelte';
// Focus aliases
export { default as FocusIcon } from '../icons/focus.svelte';
// FoldHorizontal aliases
export { default as FoldHorizontalIcon } from '../icons/fold-horizontal.svelte';
// FoldVertical aliases
export { default as FoldVerticalIcon } from '../icons/fold-vertical.svelte';
// FolderArchive aliases
export { default as FolderArchiveIcon } from '../icons/folder-archive.svelte';
// FolderCheck aliases
export { default as FolderCheckIcon } from '../icons/folder-check.svelte';
// FolderClock aliases
export { default as FolderClockIcon } from '../icons/folder-clock.svelte';
// FolderClosed aliases
export { default as FolderClosedIcon } from '../icons/folder-closed.svelte';
// FolderCode aliases
export { default as FolderCodeIcon } from '../icons/folder-code.svelte';
// FolderDot aliases
export { default as FolderDotIcon } from '../icons/folder-dot.svelte';
// FolderDown aliases
export { default as FolderDownIcon } from '../icons/folder-down.svelte';
// FolderGit2 aliases
export { default as FolderGit2Icon } from '../icons/folder-git-2.svelte';
// FolderGit aliases
export { default as FolderGitIcon } from '../icons/folder-git.svelte';
// FolderHeart aliases
export { default as FolderHeartIcon } from '../icons/folder-heart.svelte';
// FolderInput aliases
export { default as FolderInputIcon } from '../icons/folder-input.svelte';
// FolderKanban aliases
export { default as FolderKanbanIcon } from '../icons/folder-kanban.svelte';
// FolderKey aliases
export { default as FolderKeyIcon } from '../icons/folder-key.svelte';
// FolderLock aliases
export { default as FolderLockIcon } from '../icons/folder-lock.svelte';
// FolderMinus aliases
export { default as FolderMinusIcon } from '../icons/folder-minus.svelte';
// FolderOpenDot aliases
export { default as FolderOpenDotIcon } from '../icons/folder-open-dot.svelte';
// FolderOpen aliases
export { default as FolderOpenIcon } from '../icons/folder-open.svelte';
// FolderOutput aliases
export { default as FolderOutputIcon } from '../icons/folder-output.svelte';
// FolderPlus aliases
export { default as FolderPlusIcon } from '../icons/folder-plus.svelte';
// FolderRoot aliases
export { default as FolderRootIcon } from '../icons/folder-root.svelte';
// FolderSearch2 aliases
export { default as FolderSearch2Icon } from '../icons/folder-search-2.svelte';
// FolderSearch aliases
export { default as FolderSearchIcon } from '../icons/folder-search.svelte';
// FolderSymlink aliases
export { default as FolderSymlinkIcon } from '../icons/folder-symlink.svelte';
// FolderSync aliases
export { default as FolderSyncIcon } from '../icons/folder-sync.svelte';
// FolderTree aliases
export { default as FolderTreeIcon } from '../icons/folder-tree.svelte';
// FolderUp aliases
export { default as FolderUpIcon } from '../icons/folder-up.svelte';
// FolderX aliases
export { default as FolderXIcon } from '../icons/folder-x.svelte';
// Folder aliases
export { default as FolderIcon } from '../icons/folder.svelte';
// Folders aliases
export { default as FoldersIcon } from '../icons/folders.svelte';
// Footprints aliases
export { default as FootprintsIcon } from '../icons/footprints.svelte';
// Forklift aliases
export { default as ForkliftIcon } from '../icons/forklift.svelte';
// Forward aliases
export { default as ForwardIcon } from '../icons/forward.svelte';
// Frame aliases
export { default as FrameIcon } from '../icons/frame.svelte';
// Framer aliases
export { default as FramerIcon } from '../icons/framer.svelte';
// Frown aliases
export { default as FrownIcon } from '../icons/frown.svelte';
// Fuel aliases
export { default as FuelIcon } from '../icons/fuel.svelte';
// Fullscreen aliases
export { default as FullscreenIcon } from '../icons/fullscreen.svelte';
// GalleryHorizontalEnd aliases
export { default as GalleryHorizontalEndIcon } from '../icons/gallery-horizontal-end.svelte';
// GalleryHorizontal aliases
export { default as GalleryHorizontalIcon } from '../icons/gallery-horizontal.svelte';
// GalleryThumbnails aliases
export { default as GalleryThumbnailsIcon } from '../icons/gallery-thumbnails.svelte';
// GalleryVerticalEnd aliases
export { default as GalleryVerticalEndIcon } from '../icons/gallery-vertical-end.svelte';
// GalleryVertical aliases
export { default as GalleryVerticalIcon } from '../icons/gallery-vertical.svelte';
// Gamepad2 aliases
export { default as Gamepad2Icon } from '../icons/gamepad-2.svelte';
// Gamepad aliases
export { default as GamepadIcon } from '../icons/gamepad.svelte';
// Gauge aliases
export { default as GaugeIcon } from '../icons/gauge.svelte';
// Gavel aliases
export { default as GavelIcon } from '../icons/gavel.svelte';
// Gem aliases
export { default as GemIcon } from '../icons/gem.svelte';
// Ghost aliases
export { default as GhostIcon } from '../icons/ghost.svelte';
// Gift aliases
export { default as GiftIcon } from '../icons/gift.svelte';
// GitBranchPlus aliases
export { default as GitBranchPlusIcon } from '../icons/git-branch-plus.svelte';
// GitBranch aliases
export { default as GitBranchIcon } from '../icons/git-branch.svelte';
// GitCommitVertical aliases
export { default as GitCommitVerticalIcon } from '../icons/git-commit-vertical.svelte';
// GitCompareArrows aliases
export { default as GitCompareArrowsIcon } from '../icons/git-compare-arrows.svelte';
// GitCompare aliases
export { default as GitCompareIcon } from '../icons/git-compare.svelte';
// GitFork aliases
export { default as GitForkIcon } from '../icons/git-fork.svelte';
// GitGraph aliases
export { default as GitGraphIcon } from '../icons/git-graph.svelte';
// GitMerge aliases
export { default as GitMergeIcon } from '../icons/git-merge.svelte';
// GitPullRequestArrow aliases
export { default as GitPullRequestArrowIcon } from '../icons/git-pull-request-arrow.svelte';
// GitPullRequestClosed aliases
export { default as GitPullRequestClosedIcon } from '../icons/git-pull-request-closed.svelte';
// GitPullRequestCreateArrow aliases
export { default as GitPullRequestCreateArrowIcon } from '../icons/git-pull-request-create-arrow.svelte';
// GitPullRequestCreate aliases
export { default as GitPullRequestCreateIcon } from '../icons/git-pull-request-create.svelte';
// GitPullRequestDraft aliases
export { default as GitPullRequestDraftIcon } from '../icons/git-pull-request-draft.svelte';
// GitPullRequest aliases
export { default as GitPullRequestIcon } from '../icons/git-pull-request.svelte';
// Github aliases
export { default as GithubIcon } from '../icons/github.svelte';
// Gitlab aliases
export { default as GitlabIcon } from '../icons/gitlab.svelte';
// GlassWater aliases
export { default as GlassWaterIcon } from '../icons/glass-water.svelte';
// Glasses aliases
export { default as GlassesIcon } from '../icons/glasses.svelte';
// GlobeLock aliases
export { default as GlobeLockIcon } from '../icons/globe-lock.svelte';
// Globe aliases
export { default as GlobeIcon } from '../icons/globe.svelte';
// Goal aliases
export { default as GoalIcon } from '../icons/goal.svelte';
// Grab aliases
export { default as GrabIcon } from '../icons/grab.svelte';
// GraduationCap aliases
export { default as GraduationCapIcon } from '../icons/graduation-cap.svelte';
// Grape aliases
export { default as GrapeIcon } from '../icons/grape.svelte';
// Grid2x2Check aliases
export { default as Grid2x2CheckIcon } from '../icons/grid-2x2-check.svelte';
// Grid2x2X aliases
export { default as Grid2x2XIcon } from '../icons/grid-2x2-x.svelte';
// GripHorizontal aliases
export { default as GripHorizontalIcon } from '../icons/grip-horizontal.svelte';
// GripVertical aliases
export { default as GripVerticalIcon } from '../icons/grip-vertical.svelte';
// Grip aliases
export { default as GripIcon } from '../icons/grip.svelte';
// Group aliases
export { default as GroupIcon } from '../icons/group.svelte';
// Guitar aliases
export { default as GuitarIcon } from '../icons/guitar.svelte';
// Ham aliases
export { default as HamIcon } from '../icons/ham.svelte';
// Hammer aliases
export { default as HammerIcon } from '../icons/hammer.svelte';
// HandCoins aliases
export { default as HandCoinsIcon } from '../icons/hand-coins.svelte';
// HandHeart aliases
export { default as HandHeartIcon } from '../icons/hand-heart.svelte';
// HandMetal aliases
export { default as HandMetalIcon } from '../icons/hand-metal.svelte';
// HandPlatter aliases
export { default as HandPlatterIcon } from '../icons/hand-platter.svelte';
// Hand aliases
export { default as HandIcon } from '../icons/hand.svelte';
// Handshake aliases
export { default as HandshakeIcon } from '../icons/handshake.svelte';
// HardDriveDownload aliases
export { default as HardDriveDownloadIcon } from '../icons/hard-drive-download.svelte';
// HardDriveUpload aliases
export { default as HardDriveUploadIcon } from '../icons/hard-drive-upload.svelte';
// HardDrive aliases
export { default as HardDriveIcon } from '../icons/hard-drive.svelte';
// HardHat aliases
export { default as HardHatIcon } from '../icons/hard-hat.svelte';
// Hash aliases
export { default as HashIcon } from '../icons/hash.svelte';
// Haze aliases
export { default as HazeIcon } from '../icons/haze.svelte';
// HdmiPort aliases
export { default as HdmiPortIcon } from '../icons/hdmi-port.svelte';
// Heading1 aliases
export { default as Heading1Icon } from '../icons/heading-1.svelte';
// Heading2 aliases
export { default as Heading2Icon } from '../icons/heading-2.svelte';
// Heading3 aliases
export { default as Heading3Icon } from '../icons/heading-3.svelte';
// Heading4 aliases
export { default as Heading4Icon } from '../icons/heading-4.svelte';
// Heading5 aliases
export { default as Heading5Icon } from '../icons/heading-5.svelte';
// Heading6 aliases
export { default as Heading6Icon } from '../icons/heading-6.svelte';
// Heading aliases
export { default as HeadingIcon } from '../icons/heading.svelte';
// HeadphoneOff aliases
export { default as HeadphoneOffIcon } from '../icons/headphone-off.svelte';
// Headphones aliases
export { default as HeadphonesIcon } from '../icons/headphones.svelte';
// Headset aliases
export { default as HeadsetIcon } from '../icons/headset.svelte';
// HeartCrack aliases
export { default as HeartCrackIcon } from '../icons/heart-crack.svelte';
// HeartHandshake aliases
export { default as HeartHandshakeIcon } from '../icons/heart-handshake.svelte';
// HeartOff aliases
export { default as HeartOffIcon } from '../icons/heart-off.svelte';
// HeartPulse aliases
export { default as HeartPulseIcon } from '../icons/heart-pulse.svelte';
// Heart aliases
export { default as HeartIcon } from '../icons/heart.svelte';
// Heater aliases
export { default as HeaterIcon } from '../icons/heater.svelte';
// Hexagon aliases
export { default as HexagonIcon } from '../icons/hexagon.svelte';
// Highlighter aliases
export { default as HighlighterIcon } from '../icons/highlighter.svelte';
// History aliases
export { default as HistoryIcon } from '../icons/history.svelte';
// HopOff aliases
export { default as HopOffIcon } from '../icons/hop-off.svelte';
// Hop aliases
export { default as HopIcon } from '../icons/hop.svelte';
// Hospital aliases
export { default as HospitalIcon } from '../icons/hospital.svelte';
// Hotel aliases
export { default as HotelIcon } from '../icons/hotel.svelte';
// Hourglass aliases
export { default as HourglassIcon } from '../icons/hourglass.svelte';
// HousePlug aliases
export { default as HousePlugIcon } from '../icons/house-plug.svelte';
// HousePlus aliases
export { default as HousePlusIcon } from '../icons/house-plus.svelte';
// IdCard aliases
export { default as IdCardIcon } from '../icons/id-card.svelte';
// ImageDown aliases
export { default as ImageDownIcon } from '../icons/image-down.svelte';
// ImageMinus aliases
export { default as ImageMinusIcon } from '../icons/image-minus.svelte';
// ImageOff aliases
export { default as ImageOffIcon } from '../icons/image-off.svelte';
// ImagePlay aliases
export { default as ImagePlayIcon } from '../icons/image-play.svelte';
// ImagePlus aliases
export { default as ImagePlusIcon } from '../icons/image-plus.svelte';
// ImageUp aliases
export { default as ImageUpIcon } from '../icons/image-up.svelte';
// Image aliases
export { default as ImageIcon } from '../icons/image.svelte';
// Images aliases
export { default as ImagesIcon } from '../icons/images.svelte';
// Import aliases
export { default as ImportIcon } from '../icons/import.svelte';
// Inbox aliases
export { default as InboxIcon } from '../icons/inbox.svelte';
// IndianRupee aliases
export { default as IndianRupeeIcon } from '../icons/indian-rupee.svelte';
// Infinity aliases
export { default as InfinityIcon } from '../icons/infinity.svelte';
// Info aliases
export { default as InfoIcon } from '../icons/info.svelte';
// InspectionPanel aliases
export { default as InspectionPanelIcon } from '../icons/inspection-panel.svelte';
// Instagram aliases
export { default as InstagramIcon } from '../icons/instagram.svelte';
// Italic aliases
export { default as ItalicIcon } from '../icons/italic.svelte';
// IterationCcw aliases
export { default as IterationCcwIcon } from '../icons/iteration-ccw.svelte';
// IterationCw aliases
export { default as IterationCwIcon } from '../icons/iteration-cw.svelte';
// JapaneseYen aliases
export { default as JapaneseYenIcon } from '../icons/japanese-yen.svelte';
// Joystick aliases
export { default as JoystickIcon } from '../icons/joystick.svelte';
// Kanban aliases
export { default as KanbanIcon } from '../icons/kanban.svelte';
// KeyRound aliases
export { default as KeyRoundIcon } from '../icons/key-round.svelte';
// KeySquare aliases
export { default as KeySquareIcon } from '../icons/key-square.svelte';
// Key aliases
export { default as KeyIcon } from '../icons/key.svelte';
// KeyboardMusic aliases
export { default as KeyboardMusicIcon } from '../icons/keyboard-music.svelte';
// KeyboardOff aliases
export { default as KeyboardOffIcon } from '../icons/keyboard-off.svelte';
// Keyboard aliases
export { default as KeyboardIcon } from '../icons/keyboard.svelte';
// LampCeiling aliases
export { default as LampCeilingIcon } from '../icons/lamp-ceiling.svelte';
// LampDesk aliases
export { default as LampDeskIcon } from '../icons/lamp-desk.svelte';
// LampFloor aliases
export { default as LampFloorIcon } from '../icons/lamp-floor.svelte';
// LampWallDown aliases
export { default as LampWallDownIcon } from '../icons/lamp-wall-down.svelte';
// LampWallUp aliases
export { default as LampWallUpIcon } from '../icons/lamp-wall-up.svelte';
// Lamp aliases
export { default as LampIcon } from '../icons/lamp.svelte';
// LandPlot aliases
export { default as LandPlotIcon } from '../icons/land-plot.svelte';
// Landmark aliases
export { default as LandmarkIcon } from '../icons/landmark.svelte';
// Languages aliases
export { default as LanguagesIcon } from '../icons/languages.svelte';
// Laptop aliases
export { default as LaptopIcon } from '../icons/laptop.svelte';
// LassoSelect aliases
export { default as LassoSelectIcon } from '../icons/lasso-select.svelte';
// Lasso aliases
export { default as LassoIcon } from '../icons/lasso.svelte';
// Laugh aliases
export { default as LaughIcon } from '../icons/laugh.svelte';
// Layers2 aliases
export { default as Layers2Icon } from '../icons/layers-2.svelte';
// Layers3 aliases
export { default as Layers3Icon } from '../icons/layers-3.svelte';
// Layers aliases
export { default as LayersIcon } from '../icons/layers.svelte';
// LayoutDashboard aliases
export { default as LayoutDashboardIcon } from '../icons/layout-dashboard.svelte';
// LayoutGrid aliases
export { default as LayoutGridIcon } from '../icons/layout-grid.svelte';
// LayoutList aliases
export { default as LayoutListIcon } from '../icons/layout-list.svelte';
// LayoutPanelLeft aliases
export { default as LayoutPanelLeftIcon } from '../icons/layout-panel-left.svelte';
// LayoutPanelTop aliases
export { default as LayoutPanelTopIcon } from '../icons/layout-panel-top.svelte';
// LayoutTemplate aliases
export { default as LayoutTemplateIcon } from '../icons/layout-template.svelte';
// Leaf aliases
export { default as LeafIcon } from '../icons/leaf.svelte';
// LeafyGreen aliases
export { default as LeafyGreenIcon } from '../icons/leafy-green.svelte';
// Lectern aliases
export { default as LecternIcon } from '../icons/lectern.svelte';
// LetterText aliases
export { default as LetterTextIcon } from '../icons/letter-text.svelte';
// LibraryBig aliases
export { default as LibraryBigIcon } from '../icons/library-big.svelte';
// Library aliases
export { default as LibraryIcon } from '../icons/library.svelte';
// LifeBuoy aliases
export { default as LifeBuoyIcon } from '../icons/life-buoy.svelte';
// Ligature aliases
export { default as LigatureIcon } from '../icons/ligature.svelte';
// LightbulbOff aliases
export { default as LightbulbOffIcon } from '../icons/lightbulb-off.svelte';
// Lightbulb aliases
export { default as LightbulbIcon } from '../icons/lightbulb.svelte';
// Link2Off aliases
export { default as Link2OffIcon } from '../icons/link-2-off.svelte';
// Link2 aliases
export { default as Link2Icon } from '../icons/link-2.svelte';
// Link aliases
export { default as LinkIcon } from '../icons/link.svelte';
// Linkedin aliases
export { default as LinkedinIcon } from '../icons/linkedin.svelte';
// ListCheck aliases
export { default as ListCheckIcon } from '../icons/list-check.svelte';
// ListChecks aliases
export { default as ListChecksIcon } from '../icons/list-checks.svelte';
// ListCollapse aliases
export { default as ListCollapseIcon } from '../icons/list-collapse.svelte';
// ListEnd aliases
export { default as ListEndIcon } from '../icons/list-end.svelte';
// ListFilter aliases
export { default as ListFilterIcon } from '../icons/list-filter.svelte';
// ListMinus aliases
export { default as ListMinusIcon } from '../icons/list-minus.svelte';
// ListMusic aliases
export { default as ListMusicIcon } from '../icons/list-music.svelte';
// ListOrdered aliases
export { default as ListOrderedIcon } from '../icons/list-ordered.svelte';
// ListPlus aliases
export { default as ListPlusIcon } from '../icons/list-plus.svelte';
// ListRestart aliases
export { default as ListRestartIcon } from '../icons/list-restart.svelte';
// ListStart aliases
export { default as ListStartIcon } from '../icons/list-start.svelte';
// ListTodo aliases
export { default as ListTodoIcon } from '../icons/list-todo.svelte';
// ListTree aliases
export { default as ListTreeIcon } from '../icons/list-tree.svelte';
// ListVideo aliases
export { default as ListVideoIcon } from '../icons/list-video.svelte';
// ListX aliases
export { default as ListXIcon } from '../icons/list-x.svelte';
// List aliases
export { default as ListIcon } from '../icons/list.svelte';
// LoaderPinwheel aliases
export { default as LoaderPinwheelIcon } from '../icons/loader-pinwheel.svelte';
// Loader aliases
export { default as LoaderIcon } from '../icons/loader.svelte';
// LocateFixed aliases
export { default as LocateFixedIcon } from '../icons/locate-fixed.svelte';
// LocateOff aliases
export { default as LocateOffIcon } from '../icons/locate-off.svelte';
// Locate aliases
export { default as LocateIcon } from '../icons/locate.svelte';
// LockKeyhole aliases
export { default as LockKeyholeIcon } from '../icons/lock-keyhole.svelte';
// Lock aliases
export { default as LockIcon } from '../icons/lock.svelte';
// LogIn aliases
export { default as LogInIcon } from '../icons/log-in.svelte';
// LogOut aliases
export { default as LogOutIcon } from '../icons/log-out.svelte';
// Logs aliases
export { default as LogsIcon } from '../icons/logs.svelte';
// Lollipop aliases
export { default as LollipopIcon } from '../icons/lollipop.svelte';
// Luggage aliases
export { default as LuggageIcon } from '../icons/luggage.svelte';
// Magnet aliases
export { default as MagnetIcon } from '../icons/magnet.svelte';
// MailCheck aliases
export { default as MailCheckIcon } from '../icons/mail-check.svelte';
// MailMinus aliases
export { default as MailMinusIcon } from '../icons/mail-minus.svelte';
// MailOpen aliases
export { default as MailOpenIcon } from '../icons/mail-open.svelte';
// MailPlus aliases
export { default as MailPlusIcon } from '../icons/mail-plus.svelte';
// MailQuestion aliases
export { default as MailQuestionIcon } from '../icons/mail-question.svelte';
// MailSearch aliases
export { default as MailSearchIcon } from '../icons/mail-search.svelte';
// MailWarning aliases
export { default as MailWarningIcon } from '../icons/mail-warning.svelte';
// MailX aliases
export { default as MailXIcon } from '../icons/mail-x.svelte';
// Mail aliases
export { default as MailIcon } from '../icons/mail.svelte';
// Mailbox aliases
export { default as MailboxIcon } from '../icons/mailbox.svelte';
// Mails aliases
export { default as MailsIcon } from '../icons/mails.svelte';
// MapPinCheckInside aliases
export { default as MapPinCheckInsideIcon } from '../icons/map-pin-check-inside.svelte';
// MapPinCheck aliases
export { default as MapPinCheckIcon } from '../icons/map-pin-check.svelte';
// MapPinHouse aliases
export { default as MapPinHouseIcon } from '../icons/map-pin-house.svelte';
// MapPinMinusInside aliases
export { default as MapPinMinusInsideIcon } from '../icons/map-pin-minus-inside.svelte';
// MapPinMinus aliases
export { default as MapPinMinusIcon } from '../icons/map-pin-minus.svelte';
// MapPinOff aliases
export { default as MapPinOffIcon } from '../icons/map-pin-off.svelte';
// MapPinPlusInside aliases
export { default as MapPinPlusInsideIcon } from '../icons/map-pin-plus-inside.svelte';
// MapPinPlus aliases
export { default as MapPinPlusIcon } from '../icons/map-pin-plus.svelte';
// MapPinXInside aliases
export { default as MapPinXInsideIcon } from '../icons/map-pin-x-inside.svelte';
// MapPinX aliases
export { default as MapPinXIcon } from '../icons/map-pin-x.svelte';
// MapPin aliases
export { default as MapPinIcon } from '../icons/map-pin.svelte';
// MapPinned aliases
export { default as MapPinnedIcon } from '../icons/map-pinned.svelte';
// Map aliases
export { default as MapIcon } from '../icons/map.svelte';
// Martini aliases
export { default as MartiniIcon } from '../icons/martini.svelte';
// Maximize2 aliases
export { default as Maximize2Icon } from '../icons/maximize-2.svelte';
// Maximize aliases
export { default as MaximizeIcon } from '../icons/maximize.svelte';
// Medal aliases
export { default as MedalIcon } from '../icons/medal.svelte';
// MegaphoneOff aliases
export { default as MegaphoneOffIcon } from '../icons/megaphone-off.svelte';
// Megaphone aliases
export { default as MegaphoneIcon } from '../icons/megaphone.svelte';
// Meh aliases
export { default as MehIcon } from '../icons/meh.svelte';
// MemoryStick aliases
export { default as MemoryStickIcon } from '../icons/memory-stick.svelte';
// Menu aliases
export { default as MenuIcon } from '../icons/menu.svelte';
// Merge aliases
export { default as MergeIcon } from '../icons/merge.svelte';
// MessageCircleCode aliases
export { default as MessageCircleCodeIcon } from '../icons/message-circle-code.svelte';
// MessageCircleDashed aliases
export { default as MessageCircleDashedIcon } from '../icons/message-circle-dashed.svelte';
// MessageCircleHeart aliases
export { default as MessageCircleHeartIcon } from '../icons/message-circle-heart.svelte';
// MessageCircleMore aliases
export { default as MessageCircleMoreIcon } from '../icons/message-circle-more.svelte';
// MessageCircleOff aliases
export { default as MessageCircleOffIcon } from '../icons/message-circle-off.svelte';
// MessageCirclePlus aliases
export { default as MessageCirclePlusIcon } from '../icons/message-circle-plus.svelte';
// MessageCircleQuestion aliases
export { default as MessageCircleQuestionIcon } from '../icons/message-circle-question.svelte';
// MessageCircleReply aliases
export { default as MessageCircleReplyIcon } from '../icons/message-circle-reply.svelte';
// MessageCircleWarning aliases
export { default as MessageCircleWarningIcon } from '../icons/message-circle-warning.svelte';
// MessageCircleX aliases
export { default as MessageCircleXIcon } from '../icons/message-circle-x.svelte';
// MessageCircle aliases
export { default as MessageCircleIcon } from '../icons/message-circle.svelte';
// MessageSquareCode aliases
export { default as MessageSquareCodeIcon } from '../icons/message-square-code.svelte';
// MessageSquareDashed aliases
export { default as MessageSquareDashedIcon } from '../icons/message-square-dashed.svelte';
// MessageSquareDiff aliases
export { default as MessageSquareDiffIcon } from '../icons/message-square-diff.svelte';
// MessageSquareDot aliases
export { default as MessageSquareDotIcon } from '../icons/message-square-dot.svelte';
// MessageSquareHeart aliases
export { default as MessageSquareHeartIcon } from '../icons/message-square-heart.svelte';
// MessageSquareLock aliases
export { default as MessageSquareLockIcon } from '../icons/message-square-lock.svelte';
// MessageSquareMore aliases
export { default as MessageSquareMoreIcon } from '../icons/message-square-more.svelte';
// MessageSquareOff aliases
export { default as MessageSquareOffIcon } from '../icons/message-square-off.svelte';
// MessageSquarePlus aliases
export { default as MessageSquarePlusIcon } from '../icons/message-square-plus.svelte';
// MessageSquareQuote aliases
export { default as MessageSquareQuoteIcon } from '../icons/message-square-quote.svelte';
// MessageSquareReply aliases
export { default as MessageSquareReplyIcon } from '../icons/message-square-reply.svelte';
// MessageSquareShare aliases
export { default as MessageSquareShareIcon } from '../icons/message-square-share.svelte';
// MessageSquareText aliases
export { default as MessageSquareTextIcon } from '../icons/message-square-text.svelte';
// MessageSquareWarning aliases
export { default as MessageSquareWarningIcon } from '../icons/message-square-warning.svelte';
// MessageSquareX aliases
export { default as MessageSquareXIcon } from '../icons/message-square-x.svelte';
// MessageSquare aliases
export { default as MessageSquareIcon } from '../icons/message-square.svelte';
// MessagesSquare aliases
export { default as MessagesSquareIcon } from '../icons/messages-square.svelte';
// MicOff aliases
export { default as MicOffIcon } from '../icons/mic-off.svelte';
// Mic aliases
export { default as MicIcon } from '../icons/mic.svelte';
// Microchip aliases
export { default as MicrochipIcon } from '../icons/microchip.svelte';
// Microscope aliases
export { default as MicroscopeIcon } from '../icons/microscope.svelte';
// Microwave aliases
export { default as MicrowaveIcon } from '../icons/microwave.svelte';
// Milestone aliases
export { default as MilestoneIcon } from '../icons/milestone.svelte';
// MilkOff aliases
export { default as MilkOffIcon } from '../icons/milk-off.svelte';
// Milk aliases
export { default as MilkIcon } from '../icons/milk.svelte';
// Minimize2 aliases
export { default as Minimize2Icon } from '../icons/minimize-2.svelte';
// Minimize aliases
export { default as MinimizeIcon } from '../icons/minimize.svelte';
// Minus aliases
export { default as MinusIcon } from '../icons/minus.svelte';
// MonitorCheck aliases
export { default as MonitorCheckIcon } from '../icons/monitor-check.svelte';
// MonitorCog aliases
export { default as MonitorCogIcon } from '../icons/monitor-cog.svelte';
// MonitorDot aliases
export { default as MonitorDotIcon } from '../icons/monitor-dot.svelte';
// MonitorDown aliases
export { default as MonitorDownIcon } from '../icons/monitor-down.svelte';
// MonitorOff aliases
export { default as MonitorOffIcon } from '../icons/monitor-off.svelte';
// MonitorPause aliases
export { default as MonitorPauseIcon } from '../icons/monitor-pause.svelte';
// MonitorPlay aliases
export { default as MonitorPlayIcon } from '../icons/monitor-play.svelte';
// MonitorSmartphone aliases
export { default as MonitorSmartphoneIcon } from '../icons/monitor-smartphone.svelte';
// MonitorSpeaker aliases
export { default as MonitorSpeakerIcon } from '../icons/monitor-speaker.svelte';
// MonitorStop aliases
export { default as MonitorStopIcon } from '../icons/monitor-stop.svelte';
// MonitorUp aliases
export { default as MonitorUpIcon } from '../icons/monitor-up.svelte';
// MonitorX aliases
export { default as MonitorXIcon } from '../icons/monitor-x.svelte';
// Monitor aliases
export { default as MonitorIcon } from '../icons/monitor.svelte';
// MoonStar aliases
export { default as MoonStarIcon } from '../icons/moon-star.svelte';
// Moon aliases
export { default as MoonIcon } from '../icons/moon.svelte';
// MountainSnow aliases
export { default as MountainSnowIcon } from '../icons/mountain-snow.svelte';
// Mountain aliases
export { default as MountainIcon } from '../icons/mountain.svelte';
// MouseOff aliases
export { default as MouseOffIcon } from '../icons/mouse-off.svelte';
// MousePointer2 aliases
export { default as MousePointer2Icon } from '../icons/mouse-pointer-2.svelte';
// MousePointerBan aliases
export { default as MousePointerBanIcon } from '../icons/mouse-pointer-ban.svelte';
// MousePointerClick aliases
export { default as MousePointerClickIcon } from '../icons/mouse-pointer-click.svelte';
// MousePointer aliases
export { default as MousePointerIcon } from '../icons/mouse-pointer.svelte';
// Mouse aliases
export { default as MouseIcon } from '../icons/mouse.svelte';
// MoveDiagonal2 aliases
export { default as MoveDiagonal2Icon } from '../icons/move-diagonal-2.svelte';
// MoveDiagonal aliases
export { default as MoveDiagonalIcon } from '../icons/move-diagonal.svelte';
// MoveDownLeft aliases
export { default as MoveDownLeftIcon } from '../icons/move-down-left.svelte';
// MoveDownRight aliases
export { default as MoveDownRightIcon } from '../icons/move-down-right.svelte';
// MoveDown aliases
export { default as MoveDownIcon } from '../icons/move-down.svelte';
// MoveHorizontal aliases
export { default as MoveHorizontalIcon } from '../icons/move-horizontal.svelte';
// MoveLeft aliases
export { default as MoveLeftIcon } from '../icons/move-left.svelte';
// MoveRight aliases
export { default as MoveRightIcon } from '../icons/move-right.svelte';
// MoveUpLeft aliases
export { default as MoveUpLeftIcon } from '../icons/move-up-left.svelte';
// MoveUpRight aliases
export { default as MoveUpRightIcon } from '../icons/move-up-right.svelte';
// MoveUp aliases
export { default as MoveUpIcon } from '../icons/move-up.svelte';
// MoveVertical aliases
export { default as MoveVerticalIcon } from '../icons/move-vertical.svelte';
// Move aliases
export { default as MoveIcon } from '../icons/move.svelte';
// Music2 aliases
export { default as Music2Icon } from '../icons/music-2.svelte';
// Music3 aliases
export { default as Music3Icon } from '../icons/music-3.svelte';
// Music4 aliases
export { default as Music4Icon } from '../icons/music-4.svelte';
// Music aliases
export { default as MusicIcon } from '../icons/music.svelte';
// Navigation2Off aliases
export { default as Navigation2OffIcon } from '../icons/navigation-2-off.svelte';
// Navigation2 aliases
export { default as Navigation2Icon } from '../icons/navigation-2.svelte';
// NavigationOff aliases
export { default as NavigationOffIcon } from '../icons/navigation-off.svelte';
// Navigation aliases
export { default as NavigationIcon } from '../icons/navigation.svelte';
// Network aliases
export { default as NetworkIcon } from '../icons/network.svelte';
// Newspaper aliases
export { default as NewspaperIcon } from '../icons/newspaper.svelte';
// Nfc aliases
export { default as NfcIcon } from '../icons/nfc.svelte';
// NotebookPen aliases
export { default as NotebookPenIcon } from '../icons/notebook-pen.svelte';
// NotebookTabs aliases
export { default as NotebookTabsIcon } from '../icons/notebook-tabs.svelte';
// NotebookText aliases
export { default as NotebookTextIcon } from '../icons/notebook-text.svelte';
// Notebook aliases
export { default as NotebookIcon } from '../icons/notebook.svelte';
// NotepadTextDashed aliases
export { default as NotepadTextDashedIcon } from '../icons/notepad-text-dashed.svelte';
// NotepadText aliases
export { default as NotepadTextIcon } from '../icons/notepad-text.svelte';
// NutOff aliases
export { default as NutOffIcon } from '../icons/nut-off.svelte';
// Nut aliases
export { default as NutIcon } from '../icons/nut.svelte';
// OctagonMinus aliases
export { default as OctagonMinusIcon } from '../icons/octagon-minus.svelte';
// Octagon aliases
export { default as OctagonIcon } from '../icons/octagon.svelte';
// Omega aliases
export { default as OmegaIcon } from '../icons/omega.svelte';
// Option aliases
export { default as OptionIcon } from '../icons/option.svelte';
// Orbit aliases
export { default as OrbitIcon } from '../icons/orbit.svelte';
// Origami aliases
export { default as OrigamiIcon } from '../icons/origami.svelte';
// Package2 aliases
export { default as Package2Icon } from '../icons/package-2.svelte';
// PackageCheck aliases
export { default as PackageCheckIcon } from '../icons/package-check.svelte';
// PackageMinus aliases
export { default as PackageMinusIcon } from '../icons/package-minus.svelte';
// PackageOpen aliases
export { default as PackageOpenIcon } from '../icons/package-open.svelte';
// PackagePlus aliases
export { default as PackagePlusIcon } from '../icons/package-plus.svelte';
// PackageSearch aliases
export { default as PackageSearchIcon } from '../icons/package-search.svelte';
// PackageX aliases
export { default as PackageXIcon } from '../icons/package-x.svelte';
// Package aliases
export { default as PackageIcon } from '../icons/package.svelte';
// PaintBucket aliases
export { default as PaintBucketIcon } from '../icons/paint-bucket.svelte';
// PaintRoller aliases
export { default as PaintRollerIcon } from '../icons/paint-roller.svelte';
// Paintbrush aliases
export { default as PaintbrushIcon } from '../icons/paintbrush.svelte';
// Palette aliases
export { default as PaletteIcon } from '../icons/palette.svelte';
// PanelBottomClose aliases
export { default as PanelBottomCloseIcon } from '../icons/panel-bottom-close.svelte';
// PanelBottomOpen aliases
export { default as PanelBottomOpenIcon } from '../icons/panel-bottom-open.svelte';
// PanelBottom aliases
export { default as PanelBottomIcon } from '../icons/panel-bottom.svelte';
// PanelRightClose aliases
export { default as PanelRightCloseIcon } from '../icons/panel-right-close.svelte';
// PanelRightOpen aliases
export { default as PanelRightOpenIcon } from '../icons/panel-right-open.svelte';
// PanelRight aliases
export { default as PanelRightIcon } from '../icons/panel-right.svelte';
// PanelTopClose aliases
export { default as PanelTopCloseIcon } from '../icons/panel-top-close.svelte';
// PanelTopOpen aliases
export { default as PanelTopOpenIcon } from '../icons/panel-top-open.svelte';
// PanelTop aliases
export { default as PanelTopIcon } from '../icons/panel-top.svelte';
// PanelsLeftBottom aliases
export { default as PanelsLeftBottomIcon } from '../icons/panels-left-bottom.svelte';
// PanelsRightBottom aliases
export { default as PanelsRightBottomIcon } from '../icons/panels-right-bottom.svelte';
// Paperclip aliases
export { default as PaperclipIcon } from '../icons/paperclip.svelte';
// Parentheses aliases
export { default as ParenthesesIcon } from '../icons/parentheses.svelte';
// ParkingMeter aliases
export { default as ParkingMeterIcon } from '../icons/parking-meter.svelte';
// PartyPopper aliases
export { default as PartyPopperIcon } from '../icons/party-popper.svelte';
// Pause aliases
export { default as PauseIcon } from '../icons/pause.svelte';
// PawPrint aliases
export { default as PawPrintIcon } from '../icons/paw-print.svelte';
// PcCase aliases
export { default as PcCaseIcon } from '../icons/pc-case.svelte';
// PenOff aliases
export { default as PenOffIcon } from '../icons/pen-off.svelte';
// PenTool aliases
export { default as PenToolIcon } from '../icons/pen-tool.svelte';
// PencilLine aliases
export { default as PencilLineIcon } from '../icons/pencil-line.svelte';
// PencilOff aliases
export { default as PencilOffIcon } from '../icons/pencil-off.svelte';
// PencilRuler aliases
export { default as PencilRulerIcon } from '../icons/pencil-ruler.svelte';
// Pencil aliases
export { default as PencilIcon } from '../icons/pencil.svelte';
// Pentagon aliases
export { default as PentagonIcon } from '../icons/pentagon.svelte';
// Percent aliases
export { default as PercentIcon } from '../icons/percent.svelte';
// PersonStanding aliases
export { default as PersonStandingIcon } from '../icons/person-standing.svelte';
// PhilippinePeso aliases
export { default as PhilippinePesoIcon } from '../icons/philippine-peso.svelte';
// PhoneCall aliases
export { default as PhoneCallIcon } from '../icons/phone-call.svelte';
// PhoneForwarded aliases
export { default as PhoneForwardedIcon } from '../icons/phone-forwarded.svelte';
// PhoneIncoming aliases
export { default as PhoneIncomingIcon } from '../icons/phone-incoming.svelte';
// PhoneMissed aliases
export { default as PhoneMissedIcon } from '../icons/phone-missed.svelte';
// PhoneOff aliases
export { default as PhoneOffIcon } from '../icons/phone-off.svelte';
// PhoneOutgoing aliases
export { default as PhoneOutgoingIcon } from '../icons/phone-outgoing.svelte';
// Phone aliases
export { default as PhoneIcon } from '../icons/phone.svelte';
// Pi aliases
export { default as PiIcon } from '../icons/pi.svelte';
// Piano aliases
export { default as PianoIcon } from '../icons/piano.svelte';
// Pickaxe aliases
export { default as PickaxeIcon } from '../icons/pickaxe.svelte';
// PictureInPicture2 aliases
export { default as PictureInPicture2Icon } from '../icons/picture-in-picture-2.svelte';
// PictureInPicture aliases
export { default as PictureInPictureIcon } from '../icons/picture-in-picture.svelte';
// PiggyBank aliases
export { default as PiggyBankIcon } from '../icons/piggy-bank.svelte';
// PilcrowLeft aliases
export { default as PilcrowLeftIcon } from '../icons/pilcrow-left.svelte';
// PilcrowRight aliases
export { default as PilcrowRightIcon } from '../icons/pilcrow-right.svelte';
// Pilcrow aliases
export { default as PilcrowIcon } from '../icons/pilcrow.svelte';
// PillBottle aliases
export { default as PillBottleIcon } from '../icons/pill-bottle.svelte';
// Pill aliases
export { default as PillIcon } from '../icons/pill.svelte';
// PinOff aliases
export { default as PinOffIcon } from '../icons/pin-off.svelte';
// Pin aliases
export { default as PinIcon } from '../icons/pin.svelte';
// Pipette aliases
export { default as PipetteIcon } from '../icons/pipette.svelte';
// Pizza aliases
export { default as PizzaIcon } from '../icons/pizza.svelte';
// PlaneLanding aliases
export { default as PlaneLandingIcon } from '../icons/plane-landing.svelte';
// PlaneTakeoff aliases
export { default as PlaneTakeoffIcon } from '../icons/plane-takeoff.svelte';
// Plane aliases
export { default as PlaneIcon } from '../icons/plane.svelte';
// Play aliases
export { default as PlayIcon } from '../icons/play.svelte';
// Plug2 aliases
export { default as Plug2Icon } from '../icons/plug-2.svelte';
// Plug aliases
export { default as PlugIcon } from '../icons/plug.svelte';
// Plus aliases
export { default as PlusIcon } from '../icons/plus.svelte';
// PocketKnife aliases
export { default as PocketKnifeIcon } from '../icons/pocket-knife.svelte';
// Pocket aliases
export { default as PocketIcon } from '../icons/pocket.svelte';
// Podcast aliases
export { default as PodcastIcon } from '../icons/podcast.svelte';
// PointerOff aliases
export { default as PointerOffIcon } from '../icons/pointer-off.svelte';
// Pointer aliases
export { default as PointerIcon } from '../icons/pointer.svelte';
// Popcorn aliases
export { default as PopcornIcon } from '../icons/popcorn.svelte';
// Popsicle aliases
export { default as PopsicleIcon } from '../icons/popsicle.svelte';
// PoundSterling aliases
export { default as PoundSterlingIcon } from '../icons/pound-sterling.svelte';
// PowerOff aliases
export { default as PowerOffIcon } from '../icons/power-off.svelte';
// Power aliases
export { default as PowerIcon } from '../icons/power.svelte';
// Presentation aliases
export { default as PresentationIcon } from '../icons/presentation.svelte';
// PrinterCheck aliases
export { default as PrinterCheckIcon } from '../icons/printer-check.svelte';
// Printer aliases
export { default as PrinterIcon } from '../icons/printer.svelte';
// Projector aliases
export { default as ProjectorIcon } from '../icons/projector.svelte';
// Proportions aliases
export { default as ProportionsIcon } from '../icons/proportions.svelte';
// Puzzle aliases
export { default as PuzzleIcon } from '../icons/puzzle.svelte';
// Pyramid aliases
export { default as PyramidIcon } from '../icons/pyramid.svelte';
// QrCode aliases
export { default as QrCodeIcon } from '../icons/qr-code.svelte';
// Quote aliases
export { default as QuoteIcon } from '../icons/quote.svelte';
// Rabbit aliases
export { default as RabbitIcon } from '../icons/rabbit.svelte';
// Radar aliases
export { default as RadarIcon } from '../icons/radar.svelte';
// Radiation aliases
export { default as RadiationIcon } from '../icons/radiation.svelte';
// Radical aliases
export { default as RadicalIcon } from '../icons/radical.svelte';
// RadioReceiver aliases
export { default as RadioReceiverIcon } from '../icons/radio-receiver.svelte';
// RadioTower aliases
export { default as RadioTowerIcon } from '../icons/radio-tower.svelte';
// Radio aliases
export { default as RadioIcon } from '../icons/radio.svelte';
// Radius aliases
export { default as RadiusIcon } from '../icons/radius.svelte';
// RailSymbol aliases
export { default as RailSymbolIcon } from '../icons/rail-symbol.svelte';
// Rainbow aliases
export { default as RainbowIcon } from '../icons/rainbow.svelte';
// Rat aliases
export { default as RatIcon } from '../icons/rat.svelte';
// Ratio aliases
export { default as RatioIcon } from '../icons/ratio.svelte';
// ReceiptCent aliases
export { default as ReceiptCentIcon } from '../icons/receipt-cent.svelte';
// ReceiptEuro aliases
export { default as ReceiptEuroIcon } from '../icons/receipt-euro.svelte';
// ReceiptIndianRupee aliases
export { default as ReceiptIndianRupeeIcon } from '../icons/receipt-indian-rupee.svelte';
// ReceiptJapaneseYen aliases
export { default as ReceiptJapaneseYenIcon } from '../icons/receipt-japanese-yen.svelte';
// ReceiptPoundSterling aliases
export { default as ReceiptPoundSterlingIcon } from '../icons/receipt-pound-sterling.svelte';
// ReceiptRussianRuble aliases
export { default as ReceiptRussianRubleIcon } from '../icons/receipt-russian-ruble.svelte';
// ReceiptSwissFranc aliases
export { default as ReceiptSwissFrancIcon } from '../icons/receipt-swiss-franc.svelte';
// ReceiptText aliases
export { default as ReceiptTextIcon } from '../icons/receipt-text.svelte';
// Receipt aliases
export { default as ReceiptIcon } from '../icons/receipt.svelte';
// RectangleHorizontal aliases
export { default as RectangleHorizontalIcon } from '../icons/rectangle-horizontal.svelte';
// RectangleVertical aliases
export { default as RectangleVerticalIcon } from '../icons/rectangle-vertical.svelte';
// Recycle aliases
export { default as RecycleIcon } from '../icons/recycle.svelte';
// Redo2 aliases
export { default as Redo2Icon } from '../icons/redo-2.svelte';
// RedoDot aliases
export { default as RedoDotIcon } from '../icons/redo-dot.svelte';
// Redo aliases
export { default as RedoIcon } from '../icons/redo.svelte';
// RefreshCcwDot aliases
export { default as RefreshCcwDotIcon } from '../icons/refresh-ccw-dot.svelte';
// RefreshCcw aliases
export { default as RefreshCcwIcon } from '../icons/refresh-ccw.svelte';
// RefreshCwOff aliases
export { default as RefreshCwOffIcon } from '../icons/refresh-cw-off.svelte';
// RefreshCw aliases
export { default as RefreshCwIcon } from '../icons/refresh-cw.svelte';
// Refrigerator aliases
export { default as RefrigeratorIcon } from '../icons/refrigerator.svelte';
// Regex aliases
export { default as RegexIcon } from '../icons/regex.svelte';
// RemoveFormatting aliases
export { default as RemoveFormattingIcon } from '../icons/remove-formatting.svelte';
// Repeat1 aliases
export { default as Repeat1Icon } from '../icons/repeat-1.svelte';
// Repeat2 aliases
export { default as Repeat2Icon } from '../icons/repeat-2.svelte';
// Repeat aliases
export { default as RepeatIcon } from '../icons/repeat.svelte';
// ReplaceAll aliases
export { default as ReplaceAllIcon } from '../icons/replace-all.svelte';
// Replace aliases
export { default as ReplaceIcon } from '../icons/replace.svelte';
// ReplyAll aliases
export { default as ReplyAllIcon } from '../icons/reply-all.svelte';
// Reply aliases
export { default as ReplyIcon } from '../icons/reply.svelte';
// Rewind aliases
export { default as RewindIcon } from '../icons/rewind.svelte';
// Ribbon aliases
export { default as RibbonIcon } from '../icons/ribbon.svelte';
// Rocket aliases
export { default as RocketIcon } from '../icons/rocket.svelte';
// RockingChair aliases
export { default as RockingChairIcon } from '../icons/rocking-chair.svelte';
// RollerCoaster aliases
export { default as RollerCoasterIcon } from '../icons/roller-coaster.svelte';
// RotateCcwSquare aliases
export { default as RotateCcwSquareIcon } from '../icons/rotate-ccw-square.svelte';
// RotateCcw aliases
export { default as RotateCcwIcon } from '../icons/rotate-ccw.svelte';
// RotateCwSquare aliases
export { default as RotateCwSquareIcon } from '../icons/rotate-cw-square.svelte';
// RotateCw aliases
export { default as RotateCwIcon } from '../icons/rotate-cw.svelte';
// RouteOff aliases
export { default as RouteOffIcon } from '../icons/route-off.svelte';
// Route aliases
export { default as RouteIcon } from '../icons/route.svelte';
// Router aliases
export { default as RouterIcon } from '../icons/router.svelte';
// Rows4 aliases
export { default as Rows4Icon } from '../icons/rows-4.svelte';
// Rss aliases
export { default as RssIcon } from '../icons/rss.svelte';
// Ruler aliases
export { default as RulerIcon } from '../icons/ruler.svelte';
// RussianRuble aliases
export { default as RussianRubleIcon } from '../icons/russian-ruble.svelte';
// Sailboat aliases
export { default as SailboatIcon } from '../icons/sailboat.svelte';
// Salad aliases
export { default as SaladIcon } from '../icons/salad.svelte';
// Sandwich aliases
export { default as SandwichIcon } from '../icons/sandwich.svelte';
// SatelliteDish aliases
export { default as SatelliteDishIcon } from '../icons/satellite-dish.svelte';
// Satellite aliases
export { default as SatelliteIcon } from '../icons/satellite.svelte';
// SaveAll aliases
export { default as SaveAllIcon } from '../icons/save-all.svelte';
// SaveOff aliases
export { default as SaveOffIcon } from '../icons/save-off.svelte';
// Save aliases
export { default as SaveIcon } from '../icons/save.svelte';
// Scale aliases
export { default as ScaleIcon } from '../icons/scale.svelte';
// Scaling aliases
export { default as ScalingIcon } from '../icons/scaling.svelte';
// ScanBarcode aliases
export { default as ScanBarcodeIcon } from '../icons/scan-barcode.svelte';
// ScanEye aliases
export { default as ScanEyeIcon } from '../icons/scan-eye.svelte';
// ScanFace aliases
export { default as ScanFaceIcon } from '../icons/scan-face.svelte';
// ScanLine aliases
export { default as ScanLineIcon } from '../icons/scan-line.svelte';
// ScanQrCode aliases
export { default as ScanQrCodeIcon } from '../icons/scan-qr-code.svelte';
// ScanSearch aliases
export { default as ScanSearchIcon } from '../icons/scan-search.svelte';
// ScanText aliases
export { default as ScanTextIcon } from '../icons/scan-text.svelte';
// Scan aliases
export { default as ScanIcon } from '../icons/scan.svelte';
// School aliases
export { default as SchoolIcon } from '../icons/school.svelte';
// ScissorsLineDashed aliases
export { default as ScissorsLineDashedIcon } from '../icons/scissors-line-dashed.svelte';
// Scissors aliases
export { default as ScissorsIcon } from '../icons/scissors.svelte';
// ScreenShareOff aliases
export { default as ScreenShareOffIcon } from '../icons/screen-share-off.svelte';
// ScreenShare aliases
export { default as ScreenShareIcon } from '../icons/screen-share.svelte';
// ScrollText aliases
export { default as ScrollTextIcon } from '../icons/scroll-text.svelte';
// Scroll aliases
export { default as ScrollIcon } from '../icons/scroll.svelte';
// SearchCheck aliases
export { default as SearchCheckIcon } from '../icons/search-check.svelte';
// SearchCode aliases
export { default as SearchCodeIcon } from '../icons/search-code.svelte';
// SearchSlash aliases
export { default as SearchSlashIcon } from '../icons/search-slash.svelte';
// SearchX aliases
export { default as SearchXIcon } from '../icons/search-x.svelte';
// Search aliases
export { default as SearchIcon } from '../icons/search.svelte';
// Section aliases
export { default as SectionIcon } from '../icons/section.svelte';
// SendToBack aliases
export { default as SendToBackIcon } from '../icons/send-to-back.svelte';
// Send aliases
export { default as SendIcon } from '../icons/send.svelte';
// SeparatorHorizontal aliases
export { default as SeparatorHorizontalIcon } from '../icons/separator-horizontal.svelte';
// SeparatorVertical aliases
export { default as SeparatorVerticalIcon } from '../icons/separator-vertical.svelte';
// ServerCog aliases
export { default as ServerCogIcon } from '../icons/server-cog.svelte';
// ServerCrash aliases
export { default as ServerCrashIcon } from '../icons/server-crash.svelte';
// ServerOff aliases
export { default as ServerOffIcon } from '../icons/server-off.svelte';
// Server aliases
export { default as ServerIcon } from '../icons/server.svelte';
// Settings2 aliases
export { default as Settings2Icon } from '../icons/settings-2.svelte';
// Settings aliases
export { default as SettingsIcon } from '../icons/settings.svelte';
// Shapes aliases
export { default as ShapesIcon } from '../icons/shapes.svelte';
// Share2 aliases
export { default as Share2Icon } from '../icons/share-2.svelte';
// Share aliases
export { default as ShareIcon } from '../icons/share.svelte';
// Sheet aliases
export { default as SheetIcon } from '../icons/sheet.svelte';
// Shell aliases
export { default as ShellIcon } from '../icons/shell.svelte';
// ShieldAlert aliases
export { default as ShieldAlertIcon } from '../icons/shield-alert.svelte';
// ShieldBan aliases
export { default as ShieldBanIcon } from '../icons/shield-ban.svelte';
// ShieldCheck aliases
export { default as ShieldCheckIcon } from '../icons/shield-check.svelte';
// ShieldEllipsis aliases
export { default as ShieldEllipsisIcon } from '../icons/shield-ellipsis.svelte';
// ShieldHalf aliases
export { default as ShieldHalfIcon } from '../icons/shield-half.svelte';
// ShieldMinus aliases
export { default as ShieldMinusIcon } from '../icons/shield-minus.svelte';
// ShieldOff aliases
export { default as ShieldOffIcon } from '../icons/shield-off.svelte';
// ShieldPlus aliases
export { default as ShieldPlusIcon } from '../icons/shield-plus.svelte';
// ShieldQuestion aliases
export { default as ShieldQuestionIcon } from '../icons/shield-question.svelte';
// Shield aliases
export { default as ShieldIcon } from '../icons/shield.svelte';
// ShipWheel aliases
export { default as ShipWheelIcon } from '../icons/ship-wheel.svelte';
// Ship aliases
export { default as ShipIcon } from '../icons/ship.svelte';
// Shirt aliases
export { default as ShirtIcon } from '../icons/shirt.svelte';
// ShoppingBag aliases
export { default as ShoppingBagIcon } from '../icons/shopping-bag.svelte';
// ShoppingBasket aliases
export { default as ShoppingBasketIcon } from '../icons/shopping-basket.svelte';
// ShoppingCart aliases
export { default as ShoppingCartIcon } from '../icons/shopping-cart.svelte';
// Shovel aliases
export { default as ShovelIcon } from '../icons/shovel.svelte';
// ShowerHead aliases
export { default as ShowerHeadIcon } from '../icons/shower-head.svelte';
// Shrink aliases
export { default as ShrinkIcon } from '../icons/shrink.svelte';
// Shrub aliases
export { default as ShrubIcon } from '../icons/shrub.svelte';
// Shuffle aliases
export { default as ShuffleIcon } from '../icons/shuffle.svelte';
// Sigma aliases
export { default as SigmaIcon } from '../icons/sigma.svelte';
// SignalHigh aliases
export { default as SignalHighIcon } from '../icons/signal-high.svelte';
// SignalLow aliases
export { default as SignalLowIcon } from '../icons/signal-low.svelte';
// SignalMedium aliases
export { default as SignalMediumIcon } from '../icons/signal-medium.svelte';
// SignalZero aliases
export { default as SignalZeroIcon } from '../icons/signal-zero.svelte';
// Signal aliases
export { default as SignalIcon } from '../icons/signal.svelte';
// Signature aliases
export { default as SignatureIcon } from '../icons/signature.svelte';
// SignpostBig aliases
export { default as SignpostBigIcon } from '../icons/signpost-big.svelte';
// Signpost aliases
export { default as SignpostIcon } from '../icons/signpost.svelte';
// Siren aliases
export { default as SirenIcon } from '../icons/siren.svelte';
// SkipBack aliases
export { default as SkipBackIcon } from '../icons/skip-back.svelte';
// SkipForward aliases
export { default as SkipForwardIcon } from '../icons/skip-forward.svelte';
// Skull aliases
export { default as SkullIcon } from '../icons/skull.svelte';
// Slack aliases
export { default as SlackIcon } from '../icons/slack.svelte';
// Slash aliases
export { default as SlashIcon } from '../icons/slash.svelte';
// Slice aliases
export { default as SliceIcon } from '../icons/slice.svelte';
// SlidersHorizontal aliases
export { default as SlidersHorizontalIcon } from '../icons/sliders-horizontal.svelte';
// SmartphoneCharging aliases
export { default as SmartphoneChargingIcon } from '../icons/smartphone-charging.svelte';
// SmartphoneNfc aliases
export { default as SmartphoneNfcIcon } from '../icons/smartphone-nfc.svelte';
// Smartphone aliases
export { default as SmartphoneIcon } from '../icons/smartphone.svelte';
// SmilePlus aliases
export { default as SmilePlusIcon } from '../icons/smile-plus.svelte';
// Smile aliases
export { default as SmileIcon } from '../icons/smile.svelte';
// Snail aliases
export { default as SnailIcon } from '../icons/snail.svelte';
// Snowflake aliases
export { default as SnowflakeIcon } from '../icons/snowflake.svelte';
// Sofa aliases
export { default as SofaIcon } from '../icons/sofa.svelte';
// Soup aliases
export { default as SoupIcon } from '../icons/soup.svelte';
// Space aliases
export { default as SpaceIcon } from '../icons/space.svelte';
// Spade aliases
export { default as SpadeIcon } from '../icons/spade.svelte';
// Sparkle aliases
export { default as SparkleIcon } from '../icons/sparkle.svelte';
// Speaker aliases
export { default as SpeakerIcon } from '../icons/speaker.svelte';
// Speech aliases
export { default as SpeechIcon } from '../icons/speech.svelte';
// SpellCheck2 aliases
export { default as SpellCheck2Icon } from '../icons/spell-check-2.svelte';
// SpellCheck aliases
export { default as SpellCheckIcon } from '../icons/spell-check.svelte';
// Spline aliases
export { default as SplineIcon } from '../icons/spline.svelte';
// Split aliases
export { default as SplitIcon } from '../icons/split.svelte';
// SprayCan aliases
export { default as SprayCanIcon } from '../icons/spray-can.svelte';
// Sprout aliases
export { default as SproutIcon } from '../icons/sprout.svelte';
// SquareDashedBottomCode aliases
export { default as SquareDashedBottomCodeIcon } from '../icons/square-dashed-bottom-code.svelte';
// SquareDashedBottom aliases
export { default as SquareDashedBottomIcon } from '../icons/square-dashed-bottom.svelte';
// SquareRadical aliases
export { default as SquareRadicalIcon } from '../icons/square-radical.svelte';
// SquareSquare aliases
export { default as SquareSquareIcon } from '../icons/square-square.svelte';
// SquareStack aliases
export { default as SquareStackIcon } from '../icons/square-stack.svelte';
// Square aliases
export { default as SquareIcon } from '../icons/square.svelte';
// Squircle aliases
export { default as SquircleIcon } from '../icons/squircle.svelte';
// Squirrel aliases
export { default as SquirrelIcon } from '../icons/squirrel.svelte';
// Stamp aliases
export { default as StampIcon } from '../icons/stamp.svelte';
// StarHalf aliases
export { default as StarHalfIcon } from '../icons/star-half.svelte';
// StarOff aliases
export { default as StarOffIcon } from '../icons/star-off.svelte';
// Star aliases
export { default as StarIcon } from '../icons/star.svelte';
// StepBack aliases
export { default as StepBackIcon } from '../icons/step-back.svelte';
// StepForward aliases
export { default as StepForwardIcon } from '../icons/step-forward.svelte';
// Stethoscope aliases
export { default as StethoscopeIcon } from '../icons/stethoscope.svelte';
// Sticker aliases
export { default as StickerIcon } from '../icons/sticker.svelte';
// StickyNote aliases
export { default as StickyNoteIcon } from '../icons/sticky-note.svelte';
// Store aliases
export { default as StoreIcon } from '../icons/store.svelte';
// StretchHorizontal aliases
export { default as StretchHorizontalIcon } from '../icons/stretch-horizontal.svelte';
// StretchVertical aliases
export { default as StretchVerticalIcon } from '../icons/stretch-vertical.svelte';
// Strikethrough aliases
export { default as StrikethroughIcon } from '../icons/strikethrough.svelte';
// Subscript aliases
export { default as SubscriptIcon } from '../icons/subscript.svelte';
// SunDim aliases
export { default as SunDimIcon } from '../icons/sun-dim.svelte';
// SunMedium aliases
export { default as SunMediumIcon } from '../icons/sun-medium.svelte';
// SunMoon aliases
export { default as SunMoonIcon } from '../icons/sun-moon.svelte';
// SunSnow aliases
export { default as SunSnowIcon } from '../icons/sun-snow.svelte';
// Sun aliases
export { default as SunIcon } from '../icons/sun.svelte';
// Sunrise aliases
export { default as SunriseIcon } from '../icons/sunrise.svelte';
// Sunset aliases
export { default as SunsetIcon } from '../icons/sunset.svelte';
// Superscript aliases
export { default as SuperscriptIcon } from '../icons/superscript.svelte';
// SwatchBook aliases
export { default as SwatchBookIcon } from '../icons/swatch-book.svelte';
// SwissFranc aliases
export { default as SwissFrancIcon } from '../icons/swiss-franc.svelte';
// SwitchCamera aliases
export { default as SwitchCameraIcon } from '../icons/switch-camera.svelte';
// Sword aliases
export { default as SwordIcon } from '../icons/sword.svelte';
// Swords aliases
export { default as SwordsIcon } from '../icons/swords.svelte';
// Syringe aliases
export { default as SyringeIcon } from '../icons/syringe.svelte';
// Table2 aliases
export { default as Table2Icon } from '../icons/table-2.svelte';
// TableCellsMerge aliases
export { default as TableCellsMergeIcon } from '../icons/table-cells-merge.svelte';
// TableCellsSplit aliases
export { default as TableCellsSplitIcon } from '../icons/table-cells-split.svelte';
// TableColumnsSplit aliases
export { default as TableColumnsSplitIcon } from '../icons/table-columns-split.svelte';
// TableOfContents aliases
export { default as TableOfContentsIcon } from '../icons/table-of-contents.svelte';
// TableProperties aliases
export { default as TablePropertiesIcon } from '../icons/table-properties.svelte';
// TableRowsSplit aliases
export { default as TableRowsSplitIcon } from '../icons/table-rows-split.svelte';
// Table aliases
export { default as TableIcon } from '../icons/table.svelte';
// TabletSmartphone aliases
export { default as TabletSmartphoneIcon } from '../icons/tablet-smartphone.svelte';
// Tablet aliases
export { default as TabletIcon } from '../icons/tablet.svelte';
// Tablets aliases
export { default as TabletsIcon } from '../icons/tablets.svelte';
// Tag aliases
export { default as TagIcon } from '../icons/tag.svelte';
// Tags aliases
export { default as TagsIcon } from '../icons/tags.svelte';
// Tally1 aliases
export { default as Tally1Icon } from '../icons/tally-1.svelte';
// Tally2 aliases
export { default as Tally2Icon } from '../icons/tally-2.svelte';
// Tally3 aliases
export { default as Tally3Icon } from '../icons/tally-3.svelte';
// Tally4 aliases
export { default as Tally4Icon } from '../icons/tally-4.svelte';
// Tally5 aliases
export { default as Tally5Icon } from '../icons/tally-5.svelte';
// Tangent aliases
export { default as TangentIcon } from '../icons/tangent.svelte';
// Target aliases
export { default as TargetIcon } from '../icons/target.svelte';
// Telescope aliases
export { default as TelescopeIcon } from '../icons/telescope.svelte';
// TentTree aliases
export { default as TentTreeIcon } from '../icons/tent-tree.svelte';
// Tent aliases
export { default as TentIcon } from '../icons/tent.svelte';
// Terminal aliases
export { default as TerminalIcon } from '../icons/terminal.svelte';
// TestTube aliases
export { default as TestTubeIcon } from '../icons/test-tube.svelte';
// TestTubes aliases
export { default as TestTubesIcon } from '../icons/test-tubes.svelte';
// TextCursorInput aliases
export { default as TextCursorInputIcon } from '../icons/text-cursor-input.svelte';
// TextCursor aliases
export { default as TextCursorIcon } from '../icons/text-cursor.svelte';
// TextQuote aliases
export { default as TextQuoteIcon } from '../icons/text-quote.svelte';
// TextSearch aliases
export { default as TextSearchIcon } from '../icons/text-search.svelte';
// Text aliases
export { default as TextIcon } from '../icons/text.svelte';
// Theater aliases
export { default as TheaterIcon } from '../icons/theater.svelte';
// ThermometerSnowflake aliases
export { default as ThermometerSnowflakeIcon } from '../icons/thermometer-snowflake.svelte';
// ThermometerSun aliases
export { default as ThermometerSunIcon } from '../icons/thermometer-sun.svelte';
// Thermometer aliases
export { default as ThermometerIcon } from '../icons/thermometer.svelte';
// ThumbsDown aliases
export { default as ThumbsDownIcon } from '../icons/thumbs-down.svelte';
// ThumbsUp aliases
export { default as ThumbsUpIcon } from '../icons/thumbs-up.svelte';
// TicketCheck aliases
export { default as TicketCheckIcon } from '../icons/ticket-check.svelte';
// TicketMinus aliases
export { default as TicketMinusIcon } from '../icons/ticket-minus.svelte';
// TicketPercent aliases
export { default as TicketPercentIcon } from '../icons/ticket-percent.svelte';
// TicketPlus aliases
export { default as TicketPlusIcon } from '../icons/ticket-plus.svelte';
// TicketSlash aliases
export { default as TicketSlashIcon } from '../icons/ticket-slash.svelte';
// TicketX aliases
export { default as TicketXIcon } from '../icons/ticket-x.svelte';
// Ticket aliases
export { default as TicketIcon } from '../icons/ticket.svelte';
// TicketsPlane aliases
export { default as TicketsPlaneIcon } from '../icons/tickets-plane.svelte';
// Tickets aliases
export { default as TicketsIcon } from '../icons/tickets.svelte';
// TimerOff aliases
export { default as TimerOffIcon } from '../icons/timer-off.svelte';
// TimerReset aliases
export { default as TimerResetIcon } from '../icons/timer-reset.svelte';
// Timer aliases
export { default as TimerIcon } from '../icons/timer.svelte';
// ToggleLeft aliases
export { default as ToggleLeftIcon } from '../icons/toggle-left.svelte';
// ToggleRight aliases
export { default as ToggleRightIcon } from '../icons/toggle-right.svelte';
// Tornado aliases
export { default as TornadoIcon } from '../icons/tornado.svelte';
// Torus aliases
export { default as TorusIcon } from '../icons/torus.svelte';
// TouchpadOff aliases
export { default as TouchpadOffIcon } from '../icons/touchpad-off.svelte';
// Touchpad aliases
export { default as TouchpadIcon } from '../icons/touchpad.svelte';
// TowerControl aliases
export { default as TowerControlIcon } from '../icons/tower-control.svelte';
// ToyBrick aliases
export { default as ToyBrickIcon } from '../icons/toy-brick.svelte';
// Tractor aliases
export { default as TractorIcon } from '../icons/tractor.svelte';
// TrafficCone aliases
export { default as TrafficConeIcon } from '../icons/traffic-cone.svelte';
// TrainFrontTunnel aliases
export { default as TrainFrontTunnelIcon } from '../icons/train-front-tunnel.svelte';
// TrainFront aliases
export { default as TrainFrontIcon } from '../icons/train-front.svelte';
// TrainTrack aliases
export { default as TrainTrackIcon } from '../icons/train-track.svelte';
// Trash2 aliases
export { default as Trash2Icon } from '../icons/trash-2.svelte';
// Trash aliases
export { default as TrashIcon } from '../icons/trash.svelte';
// TreeDeciduous aliases
export { default as TreeDeciduousIcon } from '../icons/tree-deciduous.svelte';
// TreePine aliases
export { default as TreePineIcon } from '../icons/tree-pine.svelte';
// Trees aliases
export { default as TreesIcon } from '../icons/trees.svelte';
// Trello aliases
export { default as TrelloIcon } from '../icons/trello.svelte';
// TrendingDown aliases
export { default as TrendingDownIcon } from '../icons/trending-down.svelte';
// TrendingUpDown aliases
export { default as TrendingUpDownIcon } from '../icons/trending-up-down.svelte';
// TrendingUp aliases
export { default as TrendingUpIcon } from '../icons/trending-up.svelte';
// TriangleRight aliases
export { default as TriangleRightIcon } from '../icons/triangle-right.svelte';
// Triangle aliases
export { default as TriangleIcon } from '../icons/triangle.svelte';
// Trophy aliases
export { default as TrophyIcon } from '../icons/trophy.svelte';
// Truck aliases
export { default as TruckIcon } from '../icons/truck.svelte';
// Turtle aliases
export { default as TurtleIcon } from '../icons/turtle.svelte';
// TvMinimalPlay aliases
export { default as TvMinimalPlayIcon } from '../icons/tv-minimal-play.svelte';
// Tv aliases
export { default as TvIcon } from '../icons/tv.svelte';
// Twitch aliases
export { default as TwitchIcon } from '../icons/twitch.svelte';
// Twitter aliases
export { default as TwitterIcon } from '../icons/twitter.svelte';
// TypeOutline aliases
export { default as TypeOutlineIcon } from '../icons/type-outline.svelte';
// Type aliases
export { default as TypeIcon } from '../icons/type.svelte';
// UmbrellaOff aliases
export { default as UmbrellaOffIcon } from '../icons/umbrella-off.svelte';
// Umbrella aliases
export { default as UmbrellaIcon } from '../icons/umbrella.svelte';
// Underline aliases
export { default as UnderlineIcon } from '../icons/underline.svelte';
// Undo2 aliases
export { default as Undo2Icon } from '../icons/undo-2.svelte';
// UndoDot aliases
export { default as UndoDotIcon } from '../icons/undo-dot.svelte';
// Undo aliases
export { default as UndoIcon } from '../icons/undo.svelte';
// UnfoldHorizontal aliases
export { default as UnfoldHorizontalIcon } from '../icons/unfold-horizontal.svelte';
// UnfoldVertical aliases
export { default as UnfoldVerticalIcon } from '../icons/unfold-vertical.svelte';
// Ungroup aliases
export { default as UngroupIcon } from '../icons/ungroup.svelte';
// Unlink2 aliases
export { default as Unlink2Icon } from '../icons/unlink-2.svelte';
// Unlink aliases
export { default as UnlinkIcon } from '../icons/unlink.svelte';
// Unplug aliases
export { default as UnplugIcon } from '../icons/unplug.svelte';
// Upload aliases
export { default as UploadIcon } from '../icons/upload.svelte';
// Usb aliases
export { default as UsbIcon } from '../icons/usb.svelte';
// UserCheck aliases
export { default as UserCheckIcon } from '../icons/user-check.svelte';
// UserCog aliases
export { default as UserCogIcon } from '../icons/user-cog.svelte';
// UserMinus aliases
export { default as UserMinusIcon } from '../icons/user-minus.svelte';
// UserPen aliases
export { default as UserPenIcon } from '../icons/user-pen.svelte';
// UserPlus aliases
export { default as UserPlusIcon } from '../icons/user-plus.svelte';
// UserRoundPen aliases
export { default as UserRoundPenIcon } from '../icons/user-round-pen.svelte';
// UserRoundSearch aliases
export { default as UserRoundSearchIcon } from '../icons/user-round-search.svelte';
// UserSearch aliases
export { default as UserSearchIcon } from '../icons/user-search.svelte';
// UserX aliases
export { default as UserXIcon } from '../icons/user-x.svelte';
// User aliases
export { default as UserIcon } from '../icons/user.svelte';
// Users aliases
export { default as UsersIcon } from '../icons/users.svelte';
// UtilityPole aliases
export { default as UtilityPoleIcon } from '../icons/utility-pole.svelte';
// Variable aliases
export { default as VariableIcon } from '../icons/variable.svelte';
// Vault aliases
export { default as VaultIcon } from '../icons/vault.svelte';
// Vegan aliases
export { default as VeganIcon } from '../icons/vegan.svelte';
// VenetianMask aliases
export { default as VenetianMaskIcon } from '../icons/venetian-mask.svelte';
// VibrateOff aliases
export { default as VibrateOffIcon } from '../icons/vibrate-off.svelte';
// Vibrate aliases
export { default as VibrateIcon } from '../icons/vibrate.svelte';
// VideoOff aliases
export { default as VideoOffIcon } from '../icons/video-off.svelte';
// Video aliases
export { default as VideoIcon } from '../icons/video.svelte';
// Videotape aliases
export { default as VideotapeIcon } from '../icons/videotape.svelte';
// View aliases
export { default as ViewIcon } from '../icons/view.svelte';
// Voicemail aliases
export { default as VoicemailIcon } from '../icons/voicemail.svelte';
// Volleyball aliases
export { default as VolleyballIcon } from '../icons/volleyball.svelte';
// Volume1 aliases
export { default as Volume1Icon } from '../icons/volume-1.svelte';
// Volume2 aliases
export { default as Volume2Icon } from '../icons/volume-2.svelte';
// VolumeOff aliases
export { default as VolumeOffIcon } from '../icons/volume-off.svelte';
// VolumeX aliases
export { default as VolumeXIcon } from '../icons/volume-x.svelte';
// Volume aliases
export { default as VolumeIcon } from '../icons/volume.svelte';
// Vote aliases
export { default as VoteIcon } from '../icons/vote.svelte';
// WalletCards aliases
export { default as WalletCardsIcon } from '../icons/wallet-cards.svelte';
// Wallet aliases
export { default as WalletIcon } from '../icons/wallet.svelte';
// Wallpaper aliases
export { default as WallpaperIcon } from '../icons/wallpaper.svelte';
// Wand aliases
export { default as WandIcon } from '../icons/wand.svelte';
// Warehouse aliases
export { default as WarehouseIcon } from '../icons/warehouse.svelte';
// WashingMachine aliases
export { default as WashingMachineIcon } from '../icons/washing-machine.svelte';
// Watch aliases
export { default as WatchIcon } from '../icons/watch.svelte';
// Waves aliases
export { default as WavesIcon } from '../icons/waves.svelte';
// Waypoints aliases
export { default as WaypointsIcon } from '../icons/waypoints.svelte';
// Webcam aliases
export { default as WebcamIcon } from '../icons/webcam.svelte';
// WebhookOff aliases
export { default as WebhookOffIcon } from '../icons/webhook-off.svelte';
// Webhook aliases
export { default as WebhookIcon } from '../icons/webhook.svelte';
// Weight aliases
export { default as WeightIcon } from '../icons/weight.svelte';
// WheatOff aliases
export { default as WheatOffIcon } from '../icons/wheat-off.svelte';
// Wheat aliases
export { default as WheatIcon } from '../icons/wheat.svelte';
// WholeWord aliases
export { default as WholeWordIcon } from '../icons/whole-word.svelte';
// WifiHigh aliases
export { default as WifiHighIcon } from '../icons/wifi-high.svelte';
// WifiLow aliases
export { default as WifiLowIcon } from '../icons/wifi-low.svelte';
// WifiOff aliases
export { default as WifiOffIcon } from '../icons/wifi-off.svelte';
// WifiZero aliases
export { default as WifiZeroIcon } from '../icons/wifi-zero.svelte';
// Wifi aliases
export { default as WifiIcon } from '../icons/wifi.svelte';
// WindArrowDown aliases
export { default as WindArrowDownIcon } from '../icons/wind-arrow-down.svelte';
// Wind aliases
export { default as WindIcon } from '../icons/wind.svelte';
// WineOff aliases
export { default as WineOffIcon } from '../icons/wine-off.svelte';
// Wine aliases
export { default as WineIcon } from '../icons/wine.svelte';
// Workflow aliases
export { default as WorkflowIcon } from '../icons/workflow.svelte';
// Worm aliases
export { default as WormIcon } from '../icons/worm.svelte';
// WrapText aliases
export { default as WrapTextIcon } from '../icons/wrap-text.svelte';
// Wrench aliases
export { default as WrenchIcon } from '../icons/wrench.svelte';
// X aliases
export { default as XIcon } from '../icons/x.svelte';
// Youtube aliases
export { default as YoutubeIcon } from '../icons/youtube.svelte';
// ZapOff aliases
export { default as ZapOffIcon } from '../icons/zap-off.svelte';
// Zap aliases
export { default as ZapIcon } from '../icons/zap.svelte';
// ZoomIn aliases
export { default as ZoomInIcon } from '../icons/zoom-in.svelte';
// ZoomOut aliases
export { default as ZoomOutIcon } from '../icons/zoom-out.svelte';
// AlarmClockCheck aliases
export { default as AlarmClockCheckIcon } from '../icons/alarm-clock-check.svelte';
export { default as AlarmCheckIcon } from '../icons/alarm-clock-check.svelte';
// AlarmClockMinus aliases
export { default as AlarmClockMinusIcon } from '../icons/alarm-clock-minus.svelte';
export { default as AlarmMinusIcon } from '../icons/alarm-clock-minus.svelte';
// AlarmClockPlus aliases
export { default as AlarmClockPlusIcon } from '../icons/alarm-clock-plus.svelte';
export { default as AlarmPlusIcon } from '../icons/alarm-clock-plus.svelte';
// ArrowDown01 aliases
export { default as ArrowDown01Icon } from '../icons/arrow-down-0-1.svelte';
// ArrowDown10 aliases
export { default as ArrowDown10Icon } from '../icons/arrow-down-1-0.svelte';
// ArrowDownAZ aliases
export { default as ArrowDownAZIcon } from '../icons/arrow-down-a-z.svelte';
export { default as ArrowDownAzIcon } from '../icons/arrow-down-a-z.svelte';
// ArrowDownWideNarrow aliases
export { default as ArrowDownWideNarrowIcon } from '../icons/arrow-down-wide-narrow.svelte';
export { default as SortDescIcon } from '../icons/arrow-down-wide-narrow.svelte';
// ArrowDownZA aliases
export { default as ArrowDownZAIcon } from '../icons/arrow-down-z-a.svelte';
export { default as ArrowDownZaIcon } from '../icons/arrow-down-z-a.svelte';
// ArrowUp01 aliases
export { default as ArrowUp01Icon } from '../icons/arrow-up-0-1.svelte';
// ArrowUp10 aliases
export { default as ArrowUp10Icon } from '../icons/arrow-up-1-0.svelte';
// ArrowUpAZ aliases
export { default as ArrowUpAZIcon } from '../icons/arrow-up-a-z.svelte';
export { default as ArrowUpAzIcon } from '../icons/arrow-up-a-z.svelte';
// ArrowUpNarrowWide aliases
export { default as ArrowUpNarrowWideIcon } from '../icons/arrow-up-narrow-wide.svelte';
export { default as SortAscIcon } from '../icons/arrow-up-narrow-wide.svelte';
// ArrowUpZA aliases
export { default as ArrowUpZAIcon } from '../icons/arrow-up-z-a.svelte';
export { default as ArrowUpZaIcon } from '../icons/arrow-up-z-a.svelte';
// Axis3d aliases
export { default as Axis3dIcon } from '../icons/axis-3d.svelte';
export { default as Axis3DIcon } from '../icons/axis-3d.svelte';
// BadgeCheck aliases
export { default as BadgeCheckIcon } from '../icons/badge-check.svelte';
export { default as VerifiedIcon } from '../icons/badge-check.svelte';
// BetweenHorizontalEnd aliases
export { default as BetweenHorizontalEndIcon } from '../icons/between-horizontal-end.svelte';
export { 
/** @deprecated Renamed because of typo, use {@link BetweenHorizontalEnd} instead. This alias will be removed in v1.0 */
default as BetweenHorizonalEndIcon } from '../icons/between-horizontal-end.svelte';
// BetweenHorizontalStart aliases
export { default as BetweenHorizontalStartIcon } from '../icons/between-horizontal-start.svelte';
export { 
/** @deprecated Renamed because of typo, use {@link BetweenHorizontalStart} instead. This alias will be removed in v1.0 */
default as BetweenHorizonalStartIcon } from '../icons/between-horizontal-start.svelte';
// BookDashed aliases
export { default as BookDashedIcon } from '../icons/book-dashed.svelte';
export { default as BookTemplateIcon } from '../icons/book-dashed.svelte';
// Braces aliases
export { default as BracesIcon } from '../icons/braces.svelte';
export { default as CurlyBracesIcon } from '../icons/braces.svelte';
// Captions aliases
export { default as CaptionsIcon } from '../icons/captions.svelte';
export { default as SubtitlesIcon } from '../icons/captions.svelte';
// ChartArea aliases
export { default as ChartAreaIcon } from '../icons/chart-area.svelte';
export { 
/** @deprecated  */
default as AreaChartIcon } from '../icons/chart-area.svelte';
// ChartBarBig aliases
export { default as ChartBarBigIcon } from '../icons/chart-bar-big.svelte';
export { 
/** @deprecated  */
default as BarChartHorizontalBigIcon } from '../icons/chart-bar-big.svelte';
// ChartBar aliases
export { default as ChartBarIcon } from '../icons/chart-bar.svelte';
export { 
/** @deprecated  */
default as BarChartHorizontalIcon } from '../icons/chart-bar.svelte';
// ChartCandlestick aliases
export { default as ChartCandlestickIcon } from '../icons/chart-candlestick.svelte';
export { 
/** @deprecated  */
default as CandlestickChartIcon } from '../icons/chart-candlestick.svelte';
// ChartColumnBig aliases
export { default as ChartColumnBigIcon } from '../icons/chart-column-big.svelte';
export { 
/** @deprecated  */
default as BarChartBigIcon } from '../icons/chart-column-big.svelte';
// ChartColumnIncreasing aliases
export { default as ChartColumnIncreasingIcon } from '../icons/chart-column-increasing.svelte';
export { 
/** @deprecated  */
default as BarChart4Icon } from '../icons/chart-column-increasing.svelte';
// ChartColumn aliases
export { default as ChartColumnIcon } from '../icons/chart-column.svelte';
export { 
/** @deprecated  */
default as BarChart3Icon } from '../icons/chart-column.svelte';
// ChartLine aliases
export { default as ChartLineIcon } from '../icons/chart-line.svelte';
export { 
/** @deprecated  */
default as LineChartIcon } from '../icons/chart-line.svelte';
// ChartNoAxesColumnIncreasing aliases
export { default as ChartNoAxesColumnIncreasingIcon } from '../icons/chart-no-axes-column-increasing.svelte';
export { 
/** @deprecated  */
default as BarChartIcon } from '../icons/chart-no-axes-column-increasing.svelte';
// ChartNoAxesColumn aliases
export { default as ChartNoAxesColumnIcon } from '../icons/chart-no-axes-column.svelte';
export { 
/** @deprecated  */
default as BarChart2Icon } from '../icons/chart-no-axes-column.svelte';
// ChartNoAxesGantt aliases
export { default as ChartNoAxesGanttIcon } from '../icons/chart-no-axes-gantt.svelte';
export { 
/** @deprecated  */
default as GanttChartIcon } from '../icons/chart-no-axes-gantt.svelte';
// ChartPie aliases
export { default as ChartPieIcon } from '../icons/chart-pie.svelte';
export { 
/** @deprecated  */
default as PieChartIcon } from '../icons/chart-pie.svelte';
// ChartScatter aliases
export { default as ChartScatterIcon } from '../icons/chart-scatter.svelte';
export { 
/** @deprecated  */
default as ScatterChartIcon } from '../icons/chart-scatter.svelte';
// CircleAlert aliases
export { default as CircleAlertIcon } from '../icons/circle-alert.svelte';
export { default as AlertCircleIcon } from '../icons/circle-alert.svelte';
// CircleArrowDown aliases
export { default as CircleArrowDownIcon } from '../icons/circle-arrow-down.svelte';
export { default as ArrowDownCircleIcon } from '../icons/circle-arrow-down.svelte';
// CircleArrowLeft aliases
export { default as CircleArrowLeftIcon } from '../icons/circle-arrow-left.svelte';
export { default as ArrowLeftCircleIcon } from '../icons/circle-arrow-left.svelte';
// CircleArrowOutDownLeft aliases
export { default as CircleArrowOutDownLeftIcon } from '../icons/circle-arrow-out-down-left.svelte';
export { default as ArrowDownLeftFromCircleIcon } from '../icons/circle-arrow-out-down-left.svelte';
// CircleArrowOutDownRight aliases
export { default as CircleArrowOutDownRightIcon } from '../icons/circle-arrow-out-down-right.svelte';
export { default as ArrowDownRightFromCircleIcon } from '../icons/circle-arrow-out-down-right.svelte';
// CircleArrowOutUpLeft aliases
export { default as CircleArrowOutUpLeftIcon } from '../icons/circle-arrow-out-up-left.svelte';
export { default as ArrowUpLeftFromCircleIcon } from '../icons/circle-arrow-out-up-left.svelte';
// CircleArrowOutUpRight aliases
export { default as CircleArrowOutUpRightIcon } from '../icons/circle-arrow-out-up-right.svelte';
export { default as ArrowUpRightFromCircleIcon } from '../icons/circle-arrow-out-up-right.svelte';
// CircleArrowRight aliases
export { default as CircleArrowRightIcon } from '../icons/circle-arrow-right.svelte';
export { default as ArrowRightCircleIcon } from '../icons/circle-arrow-right.svelte';
// CircleArrowUp aliases
export { default as CircleArrowUpIcon } from '../icons/circle-arrow-up.svelte';
export { default as ArrowUpCircleIcon } from '../icons/circle-arrow-up.svelte';
// CircleCheckBig aliases
export { default as CircleCheckBigIcon } from '../icons/circle-check-big.svelte';
export { default as CheckCircleIcon } from '../icons/circle-check-big.svelte';
// CircleCheck aliases
export { default as CircleCheckIcon } from '../icons/circle-check.svelte';
export { default as CheckCircle2Icon } from '../icons/circle-check.svelte';
// CircleChevronDown aliases
export { default as CircleChevronDownIcon } from '../icons/circle-chevron-down.svelte';
export { default as ChevronDownCircleIcon } from '../icons/circle-chevron-down.svelte';
// CircleChevronLeft aliases
export { default as CircleChevronLeftIcon } from '../icons/circle-chevron-left.svelte';
export { default as ChevronLeftCircleIcon } from '../icons/circle-chevron-left.svelte';
// CircleChevronRight aliases
export { default as CircleChevronRightIcon } from '../icons/circle-chevron-right.svelte';
export { default as ChevronRightCircleIcon } from '../icons/circle-chevron-right.svelte';
// CircleChevronUp aliases
export { default as CircleChevronUpIcon } from '../icons/circle-chevron-up.svelte';
export { default as ChevronUpCircleIcon } from '../icons/circle-chevron-up.svelte';
// CircleDivide aliases
export { default as CircleDivideIcon } from '../icons/circle-divide.svelte';
export { default as DivideCircleIcon } from '../icons/circle-divide.svelte';
// CircleGauge aliases
export { default as CircleGaugeIcon } from '../icons/circle-gauge.svelte';
export { default as GaugeCircleIcon } from '../icons/circle-gauge.svelte';
// CircleHelp aliases
export { default as CircleHelpIcon } from '../icons/circle-help.svelte';
export { default as HelpCircleIcon } from '../icons/circle-help.svelte';
// CircleMinus aliases
export { default as CircleMinusIcon } from '../icons/circle-minus.svelte';
export { default as MinusCircleIcon } from '../icons/circle-minus.svelte';
// CircleParkingOff aliases
export { default as CircleParkingOffIcon } from '../icons/circle-parking-off.svelte';
export { default as ParkingCircleOffIcon } from '../icons/circle-parking-off.svelte';
// CircleParking aliases
export { default as CircleParkingIcon } from '../icons/circle-parking.svelte';
export { default as ParkingCircleIcon } from '../icons/circle-parking.svelte';
// CirclePause aliases
export { default as CirclePauseIcon } from '../icons/circle-pause.svelte';
export { default as PauseCircleIcon } from '../icons/circle-pause.svelte';
// CirclePercent aliases
export { default as CirclePercentIcon } from '../icons/circle-percent.svelte';
export { default as PercentCircleIcon } from '../icons/circle-percent.svelte';
// CirclePlay aliases
export { default as CirclePlayIcon } from '../icons/circle-play.svelte';
export { default as PlayCircleIcon } from '../icons/circle-play.svelte';
// CirclePlus aliases
export { default as CirclePlusIcon } from '../icons/circle-plus.svelte';
export { default as PlusCircleIcon } from '../icons/circle-plus.svelte';
// CirclePower aliases
export { default as CirclePowerIcon } from '../icons/circle-power.svelte';
export { default as PowerCircleIcon } from '../icons/circle-power.svelte';
// CircleSlash2 aliases
export { default as CircleSlash2Icon } from '../icons/circle-slash-2.svelte';
export { default as CircleSlashedIcon } from '../icons/circle-slash-2.svelte';
// CircleStop aliases
export { default as CircleStopIcon } from '../icons/circle-stop.svelte';
export { default as StopCircleIcon } from '../icons/circle-stop.svelte';
// CircleUserRound aliases
export { default as CircleUserRoundIcon } from '../icons/circle-user-round.svelte';
export { 
/** @deprecated  */
default as UserCircle2Icon } from '../icons/circle-user-round.svelte';
// CircleUser aliases
export { default as CircleUserIcon } from '../icons/circle-user.svelte';
export { default as UserCircleIcon } from '../icons/circle-user.svelte';
// CircleX aliases
export { default as CircleXIcon } from '../icons/circle-x.svelte';
export { default as XCircleIcon } from '../icons/circle-x.svelte';
// ClipboardPenLine aliases
export { default as ClipboardPenLineIcon } from '../icons/clipboard-pen-line.svelte';
export { default as ClipboardSignatureIcon } from '../icons/clipboard-pen-line.svelte';
// ClipboardPen aliases
export { default as ClipboardPenIcon } from '../icons/clipboard-pen.svelte';
export { default as ClipboardEditIcon } from '../icons/clipboard-pen.svelte';
// CloudDownload aliases
export { default as CloudDownloadIcon } from '../icons/cloud-download.svelte';
export { default as DownloadCloudIcon } from '../icons/cloud-download.svelte';
// CloudUpload aliases
export { default as CloudUploadIcon } from '../icons/cloud-upload.svelte';
export { default as UploadCloudIcon } from '../icons/cloud-upload.svelte';
// CodeXml aliases
export { default as CodeXmlIcon } from '../icons/code-xml.svelte';
export { 
/** @deprecated  */
default as Code2Icon } from '../icons/code-xml.svelte';
// Columns2 aliases
export { default as Columns2Icon } from '../icons/columns-2.svelte';
export { default as ColumnsIcon } from '../icons/columns-2.svelte';
// Columns3 aliases
export { default as Columns3Icon } from '../icons/columns-3.svelte';
export { default as PanelsLeftRightIcon } from '../icons/columns-3.svelte';
// ContactRound aliases
export { default as ContactRoundIcon } from '../icons/contact-round.svelte';
export { 
/** @deprecated  */
default as Contact2Icon } from '../icons/contact-round.svelte';
// DiamondPercent aliases
export { default as DiamondPercentIcon } from '../icons/diamond-percent.svelte';
export { default as PercentDiamondIcon } from '../icons/diamond-percent.svelte';
// Earth aliases
export { default as EarthIcon } from '../icons/earth.svelte';
export { 
/** @deprecated  */
default as Globe2Icon } from '../icons/earth.svelte';
// EllipsisVertical aliases
export { default as EllipsisVerticalIcon } from '../icons/ellipsis-vertical.svelte';
export { default as MoreVerticalIcon } from '../icons/ellipsis-vertical.svelte';
// Ellipsis aliases
export { default as EllipsisIcon } from '../icons/ellipsis.svelte';
export { default as MoreHorizontalIcon } from '../icons/ellipsis.svelte';
// FileAxis3d aliases
export { default as FileAxis3dIcon } from '../icons/file-axis-3d.svelte';
export { default as FileAxis3DIcon } from '../icons/file-axis-3d.svelte';
// FileChartColumnIncreasing aliases
export { default as FileChartColumnIncreasingIcon } from '../icons/file-chart-column-increasing.svelte';
export { 
/** @deprecated  */
default as FileBarChartIcon } from '../icons/file-chart-column-increasing.svelte';
// FileChartColumn aliases
export { default as FileChartColumnIcon } from '../icons/file-chart-column.svelte';
export { 
/** @deprecated  */
default as FileBarChart2Icon } from '../icons/file-chart-column.svelte';
// FileChartLine aliases
export { default as FileChartLineIcon } from '../icons/file-chart-line.svelte';
export { 
/** @deprecated  */
default as FileLineChartIcon } from '../icons/file-chart-line.svelte';
// FileChartPie aliases
export { default as FileChartPieIcon } from '../icons/file-chart-pie.svelte';
export { 
/** @deprecated  */
default as FilePieChartIcon } from '../icons/file-chart-pie.svelte';
// FileCog aliases
export { default as FileCogIcon } from '../icons/file-cog.svelte';
export { 
/** @deprecated  */
default as FileCog2Icon } from '../icons/file-cog.svelte';
// FilePenLine aliases
export { default as FilePenLineIcon } from '../icons/file-pen-line.svelte';
export { default as FileSignatureIcon } from '../icons/file-pen-line.svelte';
// FilePen aliases
export { default as FilePenIcon } from '../icons/file-pen.svelte';
export { default as FileEditIcon } from '../icons/file-pen.svelte';
// FolderCog aliases
export { default as FolderCogIcon } from '../icons/folder-cog.svelte';
export { 
/** @deprecated  */
default as FolderCog2Icon } from '../icons/folder-cog.svelte';
// FolderPen aliases
export { default as FolderPenIcon } from '../icons/folder-pen.svelte';
export { default as FolderEditIcon } from '../icons/folder-pen.svelte';
// GitCommitHorizontal aliases
export { default as GitCommitHorizontalIcon } from '../icons/git-commit-horizontal.svelte';
export { default as GitCommitIcon } from '../icons/git-commit-horizontal.svelte';
// Grid2x2Plus aliases
export { default as Grid2x2PlusIcon } from '../icons/grid-2x2-plus.svelte';
export { default as Grid2X2PlusIcon } from '../icons/grid-2x2-plus.svelte';
// Grid2x2 aliases
export { default as Grid2x2Icon } from '../icons/grid-2x2.svelte';
export { default as Grid2X2Icon } from '../icons/grid-2x2.svelte';
// Grid3x3 aliases
export { default as Grid3x3Icon } from '../icons/grid-3x3.svelte';
export { default as GridIcon } from '../icons/grid-3x3.svelte';
export { default as Grid3X3Icon } from '../icons/grid-3x3.svelte';
// HandHelping aliases
export { default as HandHelpingIcon } from '../icons/hand-helping.svelte';
export { default as HelpingHandIcon } from '../icons/hand-helping.svelte';
// House aliases
export { default as HouseIcon } from '../icons/house.svelte';
export { default as HomeIcon } from '../icons/house.svelte';
// IceCreamBowl aliases
export { default as IceCreamBowlIcon } from '../icons/ice-cream-bowl.svelte';
export { 
/** @deprecated  */
default as IceCream2Icon } from '../icons/ice-cream-bowl.svelte';
// IceCreamCone aliases
export { default as IceCreamConeIcon } from '../icons/ice-cream-cone.svelte';
export { default as IceCreamIcon } from '../icons/ice-cream-cone.svelte';
// IndentDecrease aliases
export { default as IndentDecreaseIcon } from '../icons/indent-decrease.svelte';
export { default as OutdentIcon } from '../icons/indent-decrease.svelte';
// IndentIncrease aliases
export { default as IndentIncreaseIcon } from '../icons/indent-increase.svelte';
export { default as IndentIcon } from '../icons/indent-increase.svelte';
// LaptopMinimal aliases
export { default as LaptopMinimalIcon } from '../icons/laptop-minimal.svelte';
export { 
/** @deprecated  */
default as Laptop2Icon } from '../icons/laptop-minimal.svelte';
// LoaderCircle aliases
export { default as LoaderCircleIcon } from '../icons/loader-circle.svelte';
export { default as Loader2Icon } from '../icons/loader-circle.svelte';
// LockKeyholeOpen aliases
export { default as LockKeyholeOpenIcon } from '../icons/lock-keyhole-open.svelte';
export { default as UnlockKeyholeIcon } from '../icons/lock-keyhole-open.svelte';
// LockOpen aliases
export { default as LockOpenIcon } from '../icons/lock-open.svelte';
export { default as UnlockIcon } from '../icons/lock-open.svelte';
// MicVocal aliases
export { default as MicVocalIcon } from '../icons/mic-vocal.svelte';
export { 
/** @deprecated  */
default as Mic2Icon } from '../icons/mic-vocal.svelte';
// Move3d aliases
export { default as Move3dIcon } from '../icons/move-3d.svelte';
export { default as Move3DIcon } from '../icons/move-3d.svelte';
// OctagonAlert aliases
export { default as OctagonAlertIcon } from '../icons/octagon-alert.svelte';
export { default as AlertOctagonIcon } from '../icons/octagon-alert.svelte';
// OctagonPause aliases
export { default as OctagonPauseIcon } from '../icons/octagon-pause.svelte';
export { default as PauseOctagonIcon } from '../icons/octagon-pause.svelte';
// OctagonX aliases
export { default as OctagonXIcon } from '../icons/octagon-x.svelte';
export { default as XOctagonIcon } from '../icons/octagon-x.svelte';
// PaintbrushVertical aliases
export { default as PaintbrushVerticalIcon } from '../icons/paintbrush-vertical.svelte';
export { default as Paintbrush2Icon } from '../icons/paintbrush-vertical.svelte';
// PanelBottomDashed aliases
export { default as PanelBottomDashedIcon } from '../icons/panel-bottom-dashed.svelte';
export { default as PanelBottomInactiveIcon } from '../icons/panel-bottom-dashed.svelte';
// PanelLeftClose aliases
export { default as PanelLeftCloseIcon } from '../icons/panel-left-close.svelte';
export { default as SidebarCloseIcon } from '../icons/panel-left-close.svelte';
// PanelLeftDashed aliases
export { default as PanelLeftDashedIcon } from '../icons/panel-left-dashed.svelte';
export { default as PanelLeftInactiveIcon } from '../icons/panel-left-dashed.svelte';
// PanelLeftOpen aliases
export { default as PanelLeftOpenIcon } from '../icons/panel-left-open.svelte';
export { default as SidebarOpenIcon } from '../icons/panel-left-open.svelte';
// PanelLeft aliases
export { default as PanelLeftIcon } from '../icons/panel-left.svelte';
export { default as SidebarIcon } from '../icons/panel-left.svelte';
// PanelRightDashed aliases
export { default as PanelRightDashedIcon } from '../icons/panel-right-dashed.svelte';
export { default as PanelRightInactiveIcon } from '../icons/panel-right-dashed.svelte';
// PanelTopDashed aliases
export { default as PanelTopDashedIcon } from '../icons/panel-top-dashed.svelte';
export { default as PanelTopInactiveIcon } from '../icons/panel-top-dashed.svelte';
// PanelsTopLeft aliases
export { default as PanelsTopLeftIcon } from '../icons/panels-top-left.svelte';
export { default as LayoutIcon } from '../icons/panels-top-left.svelte';
// PenLine aliases
export { default as PenLineIcon } from '../icons/pen-line.svelte';
export { 
/** @deprecated  */
default as Edit3Icon } from '../icons/pen-line.svelte';
// Pen aliases
export { default as PenIcon } from '../icons/pen.svelte';
export { 
/** @deprecated  */
default as Edit2Icon } from '../icons/pen.svelte';
// PlugZap aliases
export { default as PlugZapIcon } from '../icons/plug-zap.svelte';
export { default as PlugZap2Icon } from '../icons/plug-zap.svelte';
// RectangleEllipsis aliases
export { default as RectangleEllipsisIcon } from '../icons/rectangle-ellipsis.svelte';
export { default as FormInputIcon } from '../icons/rectangle-ellipsis.svelte';
// Rotate3d aliases
export { default as Rotate3dIcon } from '../icons/rotate-3d.svelte';
export { default as Rotate3DIcon } from '../icons/rotate-3d.svelte';
// Rows2 aliases
export { default as Rows2Icon } from '../icons/rows-2.svelte';
export { default as RowsIcon } from '../icons/rows-2.svelte';
// Rows3 aliases
export { default as Rows3Icon } from '../icons/rows-3.svelte';
export { default as PanelsTopBottomIcon } from '../icons/rows-3.svelte';
// Scale3d aliases
export { default as Scale3dIcon } from '../icons/scale-3d.svelte';
export { default as Scale3DIcon } from '../icons/scale-3d.svelte';
// SendHorizontal aliases
export { default as SendHorizontalIcon } from '../icons/send-horizontal.svelte';
export { 
/** @deprecated Renamed because of typo, use {@link SendHorizontal} instead. This alias will be removed in v1.0 */
default as SendHorizonalIcon } from '../icons/send-horizontal.svelte';
// ShieldX aliases
export { default as ShieldXIcon } from '../icons/shield-x.svelte';
export { default as ShieldCloseIcon } from '../icons/shield-x.svelte';
// SlidersVertical aliases
export { default as SlidersVerticalIcon } from '../icons/sliders-vertical.svelte';
export { default as SlidersIcon } from '../icons/sliders-vertical.svelte';
// Sparkles aliases
export { default as SparklesIcon } from '../icons/sparkles.svelte';
export { default as StarsIcon } from '../icons/sparkles.svelte';
// SquareActivity aliases
export { default as SquareActivityIcon } from '../icons/square-activity.svelte';
export { default as ActivitySquareIcon } from '../icons/square-activity.svelte';
// SquareArrowDownLeft aliases
export { default as SquareArrowDownLeftIcon } from '../icons/square-arrow-down-left.svelte';
export { default as ArrowDownLeftSquareIcon } from '../icons/square-arrow-down-left.svelte';
// SquareArrowDownRight aliases
export { default as SquareArrowDownRightIcon } from '../icons/square-arrow-down-right.svelte';
export { default as ArrowDownRightSquareIcon } from '../icons/square-arrow-down-right.svelte';
// SquareArrowDown aliases
export { default as SquareArrowDownIcon } from '../icons/square-arrow-down.svelte';
export { default as ArrowDownSquareIcon } from '../icons/square-arrow-down.svelte';
// SquareArrowLeft aliases
export { default as SquareArrowLeftIcon } from '../icons/square-arrow-left.svelte';
export { default as ArrowLeftSquareIcon } from '../icons/square-arrow-left.svelte';
// SquareArrowOutDownLeft aliases
export { default as SquareArrowOutDownLeftIcon } from '../icons/square-arrow-out-down-left.svelte';
export { default as ArrowDownLeftFromSquareIcon } from '../icons/square-arrow-out-down-left.svelte';
// SquareArrowOutDownRight aliases
export { default as SquareArrowOutDownRightIcon } from '../icons/square-arrow-out-down-right.svelte';
export { default as ArrowDownRightFromSquareIcon } from '../icons/square-arrow-out-down-right.svelte';
// SquareArrowOutUpLeft aliases
export { default as SquareArrowOutUpLeftIcon } from '../icons/square-arrow-out-up-left.svelte';
export { default as ArrowUpLeftFromSquareIcon } from '../icons/square-arrow-out-up-left.svelte';
// SquareArrowOutUpRight aliases
export { default as SquareArrowOutUpRightIcon } from '../icons/square-arrow-out-up-right.svelte';
export { default as ArrowUpRightFromSquareIcon } from '../icons/square-arrow-out-up-right.svelte';
// SquareArrowRight aliases
export { default as SquareArrowRightIcon } from '../icons/square-arrow-right.svelte';
export { default as ArrowRightSquareIcon } from '../icons/square-arrow-right.svelte';
// SquareArrowUpLeft aliases
export { default as SquareArrowUpLeftIcon } from '../icons/square-arrow-up-left.svelte';
export { default as ArrowUpLeftSquareIcon } from '../icons/square-arrow-up-left.svelte';
// SquareArrowUpRight aliases
export { default as SquareArrowUpRightIcon } from '../icons/square-arrow-up-right.svelte';
export { default as ArrowUpRightSquareIcon } from '../icons/square-arrow-up-right.svelte';
// SquareArrowUp aliases
export { default as SquareArrowUpIcon } from '../icons/square-arrow-up.svelte';
export { default as ArrowUpSquareIcon } from '../icons/square-arrow-up.svelte';
// SquareAsterisk aliases
export { default as SquareAsteriskIcon } from '../icons/square-asterisk.svelte';
export { default as AsteriskSquareIcon } from '../icons/square-asterisk.svelte';
// SquareBottomDashedScissors aliases
export { default as SquareBottomDashedScissorsIcon } from '../icons/square-bottom-dashed-scissors.svelte';
export { default as ScissorsSquareDashedBottomIcon } from '../icons/square-bottom-dashed-scissors.svelte';
// SquareChartGantt aliases
export { default as SquareChartGanttIcon } from '../icons/square-chart-gantt.svelte';
export { default as GanttChartSquareIcon } from '../icons/square-chart-gantt.svelte';
export { default as SquareGanttChartIcon } from '../icons/square-chart-gantt.svelte';
// SquareCheckBig aliases
export { default as SquareCheckBigIcon } from '../icons/square-check-big.svelte';
export { default as CheckSquareIcon } from '../icons/square-check-big.svelte';
// SquareCheck aliases
export { default as SquareCheckIcon } from '../icons/square-check.svelte';
export { default as CheckSquare2Icon } from '../icons/square-check.svelte';
// SquareChevronDown aliases
export { default as SquareChevronDownIcon } from '../icons/square-chevron-down.svelte';
export { default as ChevronDownSquareIcon } from '../icons/square-chevron-down.svelte';
// SquareChevronLeft aliases
export { default as SquareChevronLeftIcon } from '../icons/square-chevron-left.svelte';
export { default as ChevronLeftSquareIcon } from '../icons/square-chevron-left.svelte';
// SquareChevronRight aliases
export { default as SquareChevronRightIcon } from '../icons/square-chevron-right.svelte';
export { default as ChevronRightSquareIcon } from '../icons/square-chevron-right.svelte';
// SquareChevronUp aliases
export { default as SquareChevronUpIcon } from '../icons/square-chevron-up.svelte';
export { default as ChevronUpSquareIcon } from '../icons/square-chevron-up.svelte';
// SquareCode aliases
export { default as SquareCodeIcon } from '../icons/square-code.svelte';
export { default as CodeSquareIcon } from '../icons/square-code.svelte';
// SquareDashedKanban aliases
export { default as SquareDashedKanbanIcon } from '../icons/square-dashed-kanban.svelte';
export { default as KanbanSquareDashedIcon } from '../icons/square-dashed-kanban.svelte';
// SquareDashedMousePointer aliases
export { default as SquareDashedMousePointerIcon } from '../icons/square-dashed-mouse-pointer.svelte';
export { default as MousePointerSquareDashedIcon } from '../icons/square-dashed-mouse-pointer.svelte';
// SquareDashed aliases
export { default as SquareDashedIcon } from '../icons/square-dashed.svelte';
export { default as BoxSelectIcon } from '../icons/square-dashed.svelte';
// SquareDivide aliases
export { default as SquareDivideIcon } from '../icons/square-divide.svelte';
export { default as DivideSquareIcon } from '../icons/square-divide.svelte';
// SquareDot aliases
export { default as SquareDotIcon } from '../icons/square-dot.svelte';
export { default as DotSquareIcon } from '../icons/square-dot.svelte';
// SquareEqual aliases
export { default as SquareEqualIcon } from '../icons/square-equal.svelte';
export { default as EqualSquareIcon } from '../icons/square-equal.svelte';
// SquareFunction aliases
export { default as SquareFunctionIcon } from '../icons/square-function.svelte';
export { default as FunctionSquareIcon } from '../icons/square-function.svelte';
// SquareKanban aliases
export { default as SquareKanbanIcon } from '../icons/square-kanban.svelte';
export { default as KanbanSquareIcon } from '../icons/square-kanban.svelte';
// SquareLibrary aliases
export { default as SquareLibraryIcon } from '../icons/square-library.svelte';
export { default as LibrarySquareIcon } from '../icons/square-library.svelte';
// SquareM aliases
export { default as SquareMIcon } from '../icons/square-m.svelte';
export { default as MSquareIcon } from '../icons/square-m.svelte';
// SquareMenu aliases
export { default as SquareMenuIcon } from '../icons/square-menu.svelte';
export { default as MenuSquareIcon } from '../icons/square-menu.svelte';
// SquareMinus aliases
export { default as SquareMinusIcon } from '../icons/square-minus.svelte';
export { default as MinusSquareIcon } from '../icons/square-minus.svelte';
// SquareMousePointer aliases
export { default as SquareMousePointerIcon } from '../icons/square-mouse-pointer.svelte';
export { default as InspectIcon } from '../icons/square-mouse-pointer.svelte';
// SquareParkingOff aliases
export { default as SquareParkingOffIcon } from '../icons/square-parking-off.svelte';
export { default as ParkingSquareOffIcon } from '../icons/square-parking-off.svelte';
// SquareParking aliases
export { default as SquareParkingIcon } from '../icons/square-parking.svelte';
export { default as ParkingSquareIcon } from '../icons/square-parking.svelte';
// SquarePen aliases
export { default as SquarePenIcon } from '../icons/square-pen.svelte';
export { default as PenBoxIcon } from '../icons/square-pen.svelte';
export { default as EditIcon } from '../icons/square-pen.svelte';
export { default as PenSquareIcon } from '../icons/square-pen.svelte';
// SquarePercent aliases
export { default as SquarePercentIcon } from '../icons/square-percent.svelte';
export { default as PercentSquareIcon } from '../icons/square-percent.svelte';
// SquarePi aliases
export { default as SquarePiIcon } from '../icons/square-pi.svelte';
export { default as PiSquareIcon } from '../icons/square-pi.svelte';
// SquarePilcrow aliases
export { default as SquarePilcrowIcon } from '../icons/square-pilcrow.svelte';
export { default as PilcrowSquareIcon } from '../icons/square-pilcrow.svelte';
// SquarePlay aliases
export { default as SquarePlayIcon } from '../icons/square-play.svelte';
export { default as PlaySquareIcon } from '../icons/square-play.svelte';
// SquarePlus aliases
export { default as SquarePlusIcon } from '../icons/square-plus.svelte';
export { default as PlusSquareIcon } from '../icons/square-plus.svelte';
// SquarePower aliases
export { default as SquarePowerIcon } from '../icons/square-power.svelte';
export { default as PowerSquareIcon } from '../icons/square-power.svelte';
// SquareScissors aliases
export { default as SquareScissorsIcon } from '../icons/square-scissors.svelte';
export { default as ScissorsSquareIcon } from '../icons/square-scissors.svelte';
// SquareSigma aliases
export { default as SquareSigmaIcon } from '../icons/square-sigma.svelte';
export { default as SigmaSquareIcon } from '../icons/square-sigma.svelte';
// SquareSlash aliases
export { default as SquareSlashIcon } from '../icons/square-slash.svelte';
export { default as SlashSquareIcon } from '../icons/square-slash.svelte';
// SquareSplitHorizontal aliases
export { default as SquareSplitHorizontalIcon } from '../icons/square-split-horizontal.svelte';
export { default as SplitSquareHorizontalIcon } from '../icons/square-split-horizontal.svelte';
// SquareSplitVertical aliases
export { default as SquareSplitVerticalIcon } from '../icons/square-split-vertical.svelte';
export { default as SplitSquareVerticalIcon } from '../icons/square-split-vertical.svelte';
// SquareTerminal aliases
export { default as SquareTerminalIcon } from '../icons/square-terminal.svelte';
export { default as TerminalSquareIcon } from '../icons/square-terminal.svelte';
// SquareUserRound aliases
export { default as SquareUserRoundIcon } from '../icons/square-user-round.svelte';
export { 
/** @deprecated  */
default as UserSquare2Icon } from '../icons/square-user-round.svelte';
// SquareUser aliases
export { default as SquareUserIcon } from '../icons/square-user.svelte';
export { default as UserSquareIcon } from '../icons/square-user.svelte';
// SquareX aliases
export { default as SquareXIcon } from '../icons/square-x.svelte';
export { default as XSquareIcon } from '../icons/square-x.svelte';
// TestTubeDiagonal aliases
export { default as TestTubeDiagonalIcon } from '../icons/test-tube-diagonal.svelte';
export { default as TestTube2Icon } from '../icons/test-tube-diagonal.svelte';
// TextSelect aliases
export { default as TextSelectIcon } from '../icons/text-select.svelte';
export { default as TextSelectionIcon } from '../icons/text-select.svelte';
// TramFront aliases
export { default as TramFrontIcon } from '../icons/tram-front.svelte';
export { default as TrainIcon } from '../icons/tram-front.svelte';
// TreePalm aliases
export { default as TreePalmIcon } from '../icons/tree-palm.svelte';
export { default as PalmtreeIcon } from '../icons/tree-palm.svelte';
// TriangleAlert aliases
export { default as TriangleAlertIcon } from '../icons/triangle-alert.svelte';
export { default as AlertTriangleIcon } from '../icons/triangle-alert.svelte';
// TvMinimal aliases
export { default as TvMinimalIcon } from '../icons/tv-minimal.svelte';
export { 
/** @deprecated  */
default as Tv2Icon } from '../icons/tv-minimal.svelte';
// University aliases
export { default as UniversityIcon } from '../icons/university.svelte';
export { 
/** @deprecated  */
default as School2Icon } from '../icons/university.svelte';
// UserRoundCheck aliases
export { default as UserRoundCheckIcon } from '../icons/user-round-check.svelte';
export { 
/** @deprecated  */
default as UserCheck2Icon } from '../icons/user-round-check.svelte';
// UserRoundCog aliases
export { default as UserRoundCogIcon } from '../icons/user-round-cog.svelte';
export { 
/** @deprecated  */
default as UserCog2Icon } from '../icons/user-round-cog.svelte';
// UserRoundMinus aliases
export { default as UserRoundMinusIcon } from '../icons/user-round-minus.svelte';
export { 
/** @deprecated  */
default as UserMinus2Icon } from '../icons/user-round-minus.svelte';
// UserRoundPlus aliases
export { default as UserRoundPlusIcon } from '../icons/user-round-plus.svelte';
export { 
/** @deprecated  */
default as UserPlus2Icon } from '../icons/user-round-plus.svelte';
// UserRoundX aliases
export { default as UserRoundXIcon } from '../icons/user-round-x.svelte';
export { 
/** @deprecated  */
default as UserX2Icon } from '../icons/user-round-x.svelte';
// UserRound aliases
export { default as UserRoundIcon } from '../icons/user-round.svelte';
export { 
/** @deprecated  */
default as User2Icon } from '../icons/user-round.svelte';
// UsersRound aliases
export { default as UsersRoundIcon } from '../icons/users-round.svelte';
export { 
/** @deprecated  */
default as Users2Icon } from '../icons/users-round.svelte';
// UtensilsCrossed aliases
export { default as UtensilsCrossedIcon } from '../icons/utensils-crossed.svelte';
export { default as ForkKnifeCrossedIcon } from '../icons/utensils-crossed.svelte';
// Utensils aliases
export { default as UtensilsIcon } from '../icons/utensils.svelte';
export { default as ForkKnifeIcon } from '../icons/utensils.svelte';
// WalletMinimal aliases
export { default as WalletMinimalIcon } from '../icons/wallet-minimal.svelte';
export { 
/** @deprecated  */
default as Wallet2Icon } from '../icons/wallet-minimal.svelte';
// WandSparkles aliases
export { default as WandSparklesIcon } from '../icons/wand-sparkles.svelte';
export { 
/** @deprecated  */
default as Wand2Icon } from '../icons/wand-sparkles.svelte';
